\documentclass[12pt]{article}
\usepackage[utf8]{inputenc}
\usepackage{multibbl}
\usepackage{hyperref}
\usepackage{pdfpages}
\newbibliography{article}
\newbibliography{SI}

% Usage of multibbl:
% bibtex article
% bibtex SI

%\usepackage[sectionbib,numbers]{natbib}
\usepackage{eurosym}
\usepackage{float}
\usepackage{amstext,amsthm}
\usepackage{amsmath}
\usepackage{amssymb,mathtools}
\usepackage[nointegrals]{wasysym} % enthaelt iint-Def die aber auch in amsmath definiert sind
\usepackage{mathrsfs}
\usepackage{refcheck}
\usepackage{graphicx}
\usepackage[figuresleft]{rotating}
\usepackage{rotating, float, caption}


\usepackage{tikz,color,a4wide,tabularx}
\usetikzlibrary{shapes.geometric}
%\usepackage{enumerate}

\newtheorem{proposition}{Proposition}[section]
\newtheorem{corollary}[proposition]{Corollary}
\newtheorem{theorem}{Theorem}
\newtheorem{lemma}{Lemma}[section]
\theoremstyle{definition}
\newtheorem{definition}{Definition}[section]
\newtheorem{remark}{Remark}[section]
\newtheorem{example}{Example}[section]
\newtheorem{estimator}{Estimator}[section]
\pagestyle{headings}

% \setlength{\bibsep}{0pt}

\setlength{\textheight}{230mm}
\setlength{\topmargin}{-15mm} 
\setlength{\textwidth}{17cm} 
\setlength{\oddsidemargin}{-3mm}
\setlength{\columnseprule}{.1pt}
\setlength{\columnsep}{20pt}

\def\blstr{1.1}
\renewcommand{\baselinestretch}{\blstr}
\def\blstrtable{1.55}
\def\blstrcode{1.2}  
%\renewcommand{\labelenumi}{\alph{enumi})}
%%\setlength{\oddsidemargin}{0.28in} 
%%\setlength{\evensidemargin}{0.28in}

%\addtolength{\bibsep}{1.5mm}
%%\bibpunct{(}{)}{,}{a}{}{,}

%%\RequirePackage{ae,mathpple}
%%\renewcommand{\rmdefault}{ppl}
%%\renewcommand{\sfdefault}{aess}
%%\renewcommand{\ttdefault}{aett}
\newcommand{\Rpackage}[1]{{\normalfont\fontseries{b}\selectfont #1}}
\setlength{\fboxsep}{0ex}

%\usepackage{a4}
\usepackage[normalem]{ulem}
\usepackage{color}
\newcommand\crule[3][black]{\textcolor{#1}{\rule{#2}{#3}}}

\newcommand{\dirk}[3]{\textcolor{red}{\sout{#1}\textcolor{blue}{#2}}{\begin{marginpar}{\raggedright\color{red} Dirk:\tiny {#3}}\end{marginpar}}}
\newcommand{\peter}[3]{\textcolor{red}{\sout{#1}\textcolor{blue}{#2}}{\begin{marginpar}{\raggedright\color{green} Peter:\tiny {#3}}\end{marginpar}}}


\DeclareMathAlphabet{\mathpzc}{OT1}{pzc}{m}{it}
\newcommand{\smallu}{\mathpzc{u}}
\newcommand{\smallx}{\mathpzc{x}}
\newcommand{\smally}{\mathpzc{y}}
\newcommand{\smallz}{\mathpzc{z}}
\newcommand{\ind}[1]{\mathbbm{1}_{\{#1\}}} %Definition Indikatorfunktion
\newcommand{\indset}[1]{\mathbbm{1}_{#1}} %Definition Indikatorfunktion
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}

\usepackage{xcolor}
\definecolor{colAMR}{RGB}{255,0,0}
\definecolor{colAFR}{RGB}{255,191,0}
\definecolor{colEUR}{RGB}{128,255,0}
\definecolor{colEAS}{RGB}{255,0,191}
\definecolor{colSAS}{RGB}{0,255,255}
\definecolor{colMEA}{RGB}{0,64,255}
\definecolor{colOCE}{RGB}{128,0,255}
\definecolor{colSIB}{RGB}{0,255,64}



\usepackage{times}
%%\RequirePackage[T1]{fontenc}

\begin{document}
\title{\LARGE Inference of recent admixture using genotype data}

\author{\sc Peter Pfaffelhuber, Elisabeth Sester-Huss, Franz Baumdicker, \\
  \sc Jana Naue, Sabine Lutz-Bonengel, Fabian Staubach}

\date{\today}

\maketitle

\begin{abstract}
  \noindent
 The inference of biogeographic ancestry (BGA) has become a focus of forensic genetics. Misinference of BGA can have profound unwanted consequences for investigations and society. We show that recent admixture can lead to misclassification and erroneous inference of ancestry proportions, using state of the art analysis tools with (i) simulations, (ii) 1000 genomes project data, and (iii) two individuals analyzed using the ForenSeq DNA Signature Prep Kit. Subsequently, we extend existing tools for estimation of individual ancestry (IA) by allowing for different IA in both parents, leading to estimates of parental individual ancestry (PIA), and a statistical test for recent admixture. Estimation of PIA outperforms IA in most scenarios of recent admixture. Furthermore, additional information about parental ancestry can be acquired with PIA that may guide casework.
 \end{abstract}




\section{Introduction}
Inference of the biogeographical ancestry of a trace or an unknown body, using genetic markers, is a focus of recent forensic genetics research (see e.g.\ \cite{article}{Phillips2016, Eduardoff2016, Kidd2017, Bradbury2019}). Misclassification and erroneous inference of ancestry proportions can mislead casework and result in unwanted societal consequences \cite{article}{pmid28470205}. Therefore, discovering potential error sources and improving methods for BGA is essential for successful and responsible application of the technology.

{\color{blue} Throughout, we assume to have a single-source trace, i.e.\ genetic autosomal material from a single person. The inference of BGA of that person can mean one of two things: (i)} The trace is classified into one {\color{blue} (and only one)} of several groups of different origin (e.g.\ Africa, Europe, East Asia, Native America, South-East Asia and Oceania; see e.g.\ \cite{article}{Snipper2007,Pfaffelhuber2019}). {\color{blue}(ii)} It is assumed that {\color{blue} the single-source trace} consists of a mixture of ancestral genetic material originating in several {\color{blue}(continental)} groups. For the inference of such admixture proportions of individual ancestry (IA), {\sc structure} \cite{article}{Pritchard2000} and the faster {\sc admixture} have become the de-facto standards \cite{article}{Alexander2009}; see also \cite{article}{Tang2005} for the same model. {\color{blue} In this manuscript, we are mainly interested in (ii).}

To understand how recent admixture can cause errors in BGA inference, consider a recently admixed individual, i.e.\ the continental BGA of both parents differs. Since methods used in BGA classification \cite{article}{Snipper2007, Pfaffelhuber2019} can only result in single population/class label, the possibility of recent admixture is usually not even implemented by such methods, making results on classification of recently admixed individuals hard to interpret. In mixed membership models as implemented in {\sc structure} or {\sc admixture} for estimating continental (or other scales of geography) IA of a trace, the genome is thought to be a mosaic of stones of different (continental) origin. The geographic distribution of the mosaic stones (alleles) is given by the IA. Here, a main assumption is that all alleles have the same chance (given by the IA) to stem from one of the continents. However, if the BGA of the two parents differs, the chance to encounter two different alleles at a locus increases due to population differentiation, e.g. between continents \cite{article}{Wahlund1928}. This is expected to lead to a genome wide increase in heterozygosity. The violation of the assumption of equal chances for homologous allelic states could potentially lead to misinferences. Given that recent admixture is a common issue in the light of increased human mobility, currently and in the past decades, such misinference could be a common error source in forensic genetic analyses. Therefore, a more comprehensive understanding of the consequences and frequency of recent admixture in forensic analyses is needed.

For forensic applications the approaches taken by Zou et al. (2015) \cite{article}{Zhou2015} and Pei et al. (2020) \cite{article}{Pei2020} to infer recent admixture are often not feasible because they rely on the inference of phase along the chromosome with dense marker sets. Furthermore, these approaches are computationally demanding. Crouch and Weale (2012) developed the LEAPFrOG algorithm that can be used with the limited marker density of most forensic applications \cite{article}{Crouch2012}. These authors inferred the parental IA with a maximum likelihood approach and applied their method to simulated and forensically relevant datasets with a focus on European/African admixture. To identify recently admixed individuals in forensic samples, an excess of heterozygous sites was used in statistical tests \cite{article}{McNevin2019}. Tvedebrink et al. (2018) and Tvedebrink and Eriksen (2019) developed likelihood-ratio tests for the null-hypothesis of non-admixture and recent (first generation) admixture, respectively, versus the alternative that the studied sample is not represented in the reference database \cite{article}{Tvedebrink2018, Tvedebrink2019}.

What is currently missing is an approach to test for recent admixture in a trace, where the null-hypothesis is that both parents have the same IA. The alternative hypothesis, called the recent-admixture model below, would be that the studied sample has parents of different IA and therefore shows recent-admixture of populations within the reference database. If such an approach also identified the IAs of the parents of an admixed individual, this might inform casework. Moreover, for a better understanding of the potential misinference in forensic applications, simulations should be based on the most realistic human population genetic models and include the analysis of recent methods and marker sets. Given global mobility of humans, global sample collections should be included in the analyses.

Our goals were to identify and quantify potential errors that result from recent admixture in standard methods for BGA inference and to develop a statistical test for recent admixture vs the null hypothesis of non-admixture. To this end, we developed a method for the inference of admixture proportions of both parents (parental individual ancestry, PIA). We use this method to (i) assign IA to the parents, (ii) improve current methods of ancestry inference, (iii) perform a likelihood ratio test to identify recently admixed individuals{\color{blue}, including the report of a $p$-value for the null hypothesis of no recent admixture}. For assessing our method and also to assess the misinference of BGA with standard methods in the context of recent admixture, we leverage population genetic simulations including the most realistic human population genetic scenarios. Furthermore, we {\color{blue} analyse} a global dataset from the 1000 genomes project, and two {\color{blue} additional }samples using the ForenSeq Signature Prep Kit on a MiSeq FGx (Verogen). 

\section{Materials and Methods}
\subsection{The admixture and recent admixture model}
We start by briefly recalling the admixture model, which is the basis for the widely used sofware {\sc structure} \cite{article}{Pritchard2000}, {\sc admixture} \cite{article}{Alexander2009}, {\sc frappe} \cite{article}{Tang2005} {\color{blue}and the {\sc R}-package {\sc LEA} \cite{article}{LEA2015}}. Afterwards, we introduce a new model, called the recent-admixture model. More details on the derivations in the admixture and recent-admixture model can be found in the SI. Moreover, the implementation of our methods can be downloaded from \url{https://github.com/pfaffelh/recent-admixture}. 

{\color{blue} Here comes an informal description of the models and the inference procedure.} For both, the admixture and recent-admixture model, we assume to have a reference database of $M$ bi-allelic markers from $K$ populations. However, from this reference database, we only need to know allele frequencies, i.e.\  $p_{mk}$ is the frequency of allele~1 (or reference allele) at marker $m$ in population $k$ for all $m=1,...,M$ and $k=1,...,K$. We have a trace with $G_m \in \{0,1,2\}$ copies of allele~1 at marker $m$ for $m=1,...,M${\color{blue} and we denote the full trace by $G = (G_m)_{m=1,...,M}$}. {\color{blue} The admixture model assumes that a fraction $q_k$ (to be estimated) of the genome of the trace comes from population $k$ for $k=1,...,K$. (We will denote $q = (q_1,...,q_K)$ the individual admixture (IA) of the trace.) Estimation of $q$ is done via maximum likelihood, where linkage equilibrium within populations of all markers is assumed; see \eqref{eqSI:logLadmixture} for the corresponding log-likelihood-function. In the recent-admixture model, we assume that the trace originates from two parents which have (potentially) different IAs, $q^M$ and $q^P$, denoted parental individual admixture (PIA) of the trace. The upshot is that the two allelic copies of the trace do not segregate independently, since each parent gave one copy {\color{blue} to their child causing} the trace. Again, this leads to a likelihood; see \eqref{eqSI:logLrecentadmixture}. Since the two models are nested (the admixture model is the special case $q^M = q^P$ in the recent-admixture model), we can subsequently use the likelihood-ratio-test framework for a formal test.} {\color{blue} While superscripts $M$ and $P$ indicate maternal and paternal ancestry, we note that the estimation of $q^M$ and $q^P$ separately is not possible since we are only dealing with autosomal markers. In terms of our method, the likelihood is symmetric in $q^M$ and $q^P$, so we can only estimate IA of the parents irrespective which is paternal and which is maternal. }

{\color{blue}Briefly, we collect the main assumptions of both models, based  exclusively on autosomal data: The ancestral populations are assumed to be homogeneous in terms of allele frequencies. Markers are assumed to be in linkage equilibrium within populations. For the trace in the admixture model (its parents in the recent-admixture model), we assume that each allele (from the parents of the trace) has the same independent chance to come from one of the ancestral groups.}

We will assume throughout that the allele frequencies $p_{mk}$ are given and will not be changed by analysing the trace. This is important since in currently used software {\sc  structure}, {\sc admixuture} and {\sc frappe}, mostly in non-forensic use, it is frequently the case that many new individuals are studied, and allele frequencies are updated. For forensic use, when analysing several traces at once, this would imply that the results for the ancestry of trace~1 depend not only on the reference data, but also on the data for traces~2, 3,... which seems inappropriate. Hence, we do not make the computational overload of updating allele frequencies, which would also lead to increased runtimes. Instead, we take the allele frequencies as given in the reference database. This approach has also been used in early papers such as  \cite{article}{Chakraborty1986} and \cite{article}{Hanis1986}.

%\subsection{The admixture model}
%Assuming that each allele observed in the trace comes from population $k$ with probability $q_k$ {\color{blue} (note that this implies $\sum_{k=1}^K q_k = 1$)}, the probability to observe allele~1 {\color{blue}or reference allele} at marker $m$ is {\color{blue}(for each of the two copies in a diploid individual)}
%\begin{align}
%  \label{eq:beta}
%  \beta_m(q) := \sum_k p_{mk} q_k,  
%\end{align}
%and the log-likelihood of $q = (q_k)_{k=1,...,K}$ is{\color{blue}, based on the diploid trace genome $G = (G_m)_{m=1,...,M}$, is readily computed} (see~\eqref{eqSI:logLadmixture} in the SI).
%\begin{align}\label{eq:logLadmixture}
%  \ell(q|G) = \sum_{m=1}^M \log\Big(\binom{2}{G_m} \beta_m(q)^{G_m}(1-\beta_m(q))^{2-G_m}\Big).
%\end{align}
%Assuming that all $p_{mk}$'s are known, this function can be maximized
%over $q$ by {\color{blue} solving a fixed point equation; see \eqref{eqSI:fixed} in the SI.} 
%computing $\hat q = (\hat q_k)_{k=1,...,K}$ such that
%$\hat q_k = f_k(\hat q)$ for (see also \eqref{eqSI:fixed} in the SI)
%\begin{align}\label{eq:fixed}
%  f_k(q) =
%  \frac{1}{2M} \sum_{m=1}^M \Big(G_m %\frac{p_{mk}}{\beta_m(q)} + %(2-G_m)\frac{1-p_{mk}}{1-\beta_m(q)}\Big)q_k,
%  \qquad k =1,...,K.
%\end{align}
%This can be done numerically by iterating
%$q_{n+1} = (f_k(q_n))_{k=1,...,K}$ until convergence. ({\color{blue} Note, however, that we do not have a formal proof of convergence for this iteration. At least, in our numerical experiments, convergence  -- in the sense that we continue the iteration until $|q_{n+1} - q_n|< 10^{-6}$ -- always happened.}) We note that this approach is essentially the same as in the EM-algorithm from \cite{article}{Tang2005}, but combining the expectation and maximization steps, since we do not update allele frequencies. In addition, although maximizing \eqref{eq:logLadmixture} could also be handled using a Newton method as in \cite{article}{Alexander2009}, this approach has the advantage that $q_n$'s are positive in all steps, and the sum of all entries in $q_n$ is always~1. Moreover, the iteration is computationally fast if only a small or moderate number of {\color{blue} markers} is considered.

%\subsection{The recent-admixture model}
%{\color{blue}Here, we assume that} mother and father of an individual come with their own vectors of admixture proportions, $q^M$ and $q^P$. {\color{blue} In this model, the log-likelihood takes into account that the two copies of each allele have different chances, as given by $\beta_m(q^M)$ and $\beta_m(q^P)$ to coincide with the reference allele;}
%(see \eqref{eqSI:logLrecentadmixture} in the SI).
 
\subsection{Obtaining admixed individuals in silico}
\label{S:insilico}
{\color{blue}In the sequel, we call an individual anciently admixed if its IA satisfies $q = q^M = q^P$, and recently admixed if $q^M\neq q^P$. If we are given allele frequencies $p_{mk}$ for all markers $m=1,...,M$ in all populations $k=1,...,K$, we can obtain an anciently admixed individual with IA given by $q$ by choosing the genome $G=(G_m)_{m=1,...,M}$ with independent markers with $G_m \sim B(2, \beta_m(q))$, the binomial distribution with two trials and success probability $\beta_m(q)$. In addition, if we are given two genomes $\tilde G = (\tilde G_m)_{m=1,...,M}$ and $\bar G = (\bar G_m)_{m=1,...,M}$ with different IA, we can obtain a recently admixed genome $G=(G_m)_{m=1,...,M}$ by setting $G_m = X_m + Y_m$ with $X_m \sim B(1, \tilde G_m/2)$ and $Y_M \sim B(1, \bar G_m/2)$. }%In order to test our method, we created in silico recently admixed individuals from a reference database. For example, we obtain an individual admixed from populations $k$ and $k'$ by choosing a genome $\tilde G = (\tilde G_m)_{m=1,...,M}$ from population $k$ and $\bar G = (\bar G_m)_{m=1,...,M}$ from population $k'$ as the parents. Then, $(G_m)_{m=1,...,M}$ are independent with $G_m = X_m + Y_m$, where $X_m = 1$ with probability $\tilde G_m/2$, $X_m=0$ with probability $1 - \tilde G_m/2$ and $Y_m = 1$ with probability $\bar G_m/2$, $Y_m=0$ with probability $1 - \bar G_m/2$. 
When iterating this procedure, we can also model second-generation admixed individuals etc.\ in silico.

{\color{blue} Consider data in a reference database (e.g.\ a subset of the 1000 genomes dataset, see Section~\ref{ss:1000}),}  consisting of Africans (AFR), East-Asians (EAS), Europeans (EUR) and South-East-Asians (SAS). All cases for second generation admixed individuals fall into one of seven categories. Writing up the ancestries of the four grandparents {\it Mother of mother/father of mother$\times$mother of father/father of father}, we have the following distinguishable cases for second generation admixed individuals (the full list of all resulting 55 cases is given in {\color{blue} Section S2.2 in} the SI; note that \cite{article}{Cheung2018} come up with only 35 cases, since they do not distinguish between maternal and paternal ancestry, e.g.\ they count AFR/AFR$\times$EAS/EAS and AFR/EAS$\times$AFR/EAS as one case):

\begin{itemize}
\item[(A)] 4 non-admixed cases, e.g.\ AFR/AFR$\times$AFR/AFR;
\item[(B)] 6 admixed cases with admixture ratio 50:50, where both parents are non-admixed, e.g. AFR/AFR$\times$EAS/EAS;
\item[(C)] 6 admixed cases with admixture ratio 50:50, where both parents are admixed, e.g.\ AFR/EAS$\times$AFR/EAS;
\item[(D)] 12 admixed cases with admixture ratio 75:25, e.g.\ AFR/AFR$\times$AFR/EAS;
\item[(E)] 12 admixed cases with admixture ratio 50:25:25, where one parent is non-admixed, e.g.\ AFR/AFR$\times$EAS/EUR;
\item[(F)] 12 admixed with admixture ratio 50:25:25, where both parents are admixed, e.g.\ AFR/EAS$\times$AFR/EUR;
\item[(G)] 3 admixed cases with admixture ratio 25:25:25:25, e.g.\ AFR/EAS$\times$EUR/SAS;
\end{itemize}
For each of these 55 cases, we simulated 500 individuals in silico by picking four grandparents at random, creating mother and father from the grandparents, and creating a new individual from the parents, as described above.

\subsection{Comparing results from admixture and recent-admixture}
For a reference database from which we compute (or estimate) allele frequencies $p_{mk}$, we can {\color{blue} compute the} estimate {\color{blue}$\hat q$} from the admixture model as well as {\color{blue} $\hat q^M, \hat q^P$} from the recent-admixture model, as described in \eqref{eqSI:fixed} and \eqref{eqSI:fixedRecent}. In order to compare the results from the admixture and recent-admixture model, we use deviations from {\color{blue} the true} $q = (q_k)_{k=1,...,K}$, where $q_k = 1$ for a non-admixed individual from population $k$, $q_k =q_{k'} = 0.5$ for an admixed individual with parents from populations $k$ and $k'$, etc. We {\color{blue}follow \cite{article}{Cheung2018} and use the divergence measures} for the admixture model
\begin{align}
  \label{eq:error}
  \sum_k |\hat q_k - q_k| \text{  and }\sum_k |\tfrac 12 (\hat q_k^{M} + \hat q_k^{P}) - q_k|
\end{align}
for the recent-admixture model, respectively.  {\color{blue}For estimating IA by using $\tfrac 12 (\hat q^{M} + \hat q^{P})$, the recent-admixture model comes with the same number of model parameters as the admixture model.} However, we stress that in the recent-admixture model, we obtain results for $q^M$ and $q^P$ separately, such that even more information than $\tfrac 12 (q^{M} + q^{P})$ is contained in the estimates for this model.


\subsection{Likelihood ratio {\color{blue}test} for recent admixture}
\label{ss:LR}
We want to see if data $G = (G_m)_{m=1,...,M}$ from a new trace fits significantly better to the recent-admixture model than to the admixture model. Since the admixture model is identical to the recent-admixture model for $q = q^M = q^P$, this amounts to a likelihood ratio test of $H_0: q^M = q^P$ against $H_1: q^M\neq q^P$. For this, we take the estimators $\hat q$ of $q$ from iteration of \eqref{eqSI:fixed}, and $\hat q^M, \hat q^P$ of $q^M$ and $q^P$ from iteration of \eqref{eqSI:fixedRecent} and compute
\begin{align}
  \label{eq:DeltaEll}
  \Delta\ell := \ell(\hat q^M, \hat q^P|G) - \ell(\hat q|G)
\end{align}
with $\ell(q^M, q^P|G)$ from \eqref{eqSI:logLrecentadmixture} and $\ell(q|G)$ from \eqref{eqSI:logLadmixture}. As usual in
likelihood ratio tests, if $\Delta \ell >x$ for some $x$ (which needs to be specified), the recent-admixture model fits significantly better and we reject $H_0$. If $\Delta \ell \leq x$, we accept $H_0$. Since markers are assumed to segregate independently, both $\ell(\hat q|G)$ and $\ell(\hat q^M, \hat q^P|G)$ are sums over all $M$ markers, such that we can even report the contribution of every AIM to $\Delta\ell$. 

{\color{blue} In an applied context, it is important to obtain a $p$-value from an observed value of $\Delta\ell$. The standard asymptotical theory  for LR-tests (for large $M$) suggests that -- under $H_0$ -- $\Delta\ell$ is $\chi^2$-distributed with $K-1$
degrees of freedom. However, in our applications, the number of markers is limited such that the asymptotic theory might not apply. Moreover, we see that the $\chi^2$-distribution turns out to be (i) too conservative in many cases and (ii) does not correctly account for the dependence of $\Delta\ell$ on $q$; see Figure~\ref{fig:qvalues} in the SI. Therefore,  we rely on simulations for obtaining a $p$-value in the following ideal way: First, we compute $\hat q$ as well as $(\hat q^M, \hat q^P)$. Then, we simulate individuals under $H_0$, i.e.\ anciently admixed individuals with markers segregating independently and IA given by $\hat q$, and compute the corresponding $\Delta\ell$-values for these individuals. Then, the $p$-value of the trace is the frequency of individuals with a higher $\Delta\ell$ than the $\Delta\ell$-value of the trace. At least if $\hat q$ is close to $q$, this gives uniformly distributed $p$-values, which every statistical test aims for. However, we recognized that there are two factors confounding these reported $p$-values. First, in non-admixed traces, there is a chance that $\hat q^M \approx \hat q^P$ and small $\Delta\ell$, but still leading to a significant result because $\Delta\ell$ is even smaller in most simulated samples. Since $\hat q^M \approx \hat q^P$ in these cases, recent admixture is not a useful conclusion, so we alter the procedure and report a $p$-value of 1 if $|\hat q^M - \hat q^P| < 0.1$, where $|.|$ is given by \eqref{eq:error}. Second, in anciently admixed individuals originating from two or more populations, we overestimate the $p$-value by a factor at most two; see Section~\ref{ss:pvalues} below. Therefore, we alter the reported $p$-value and multiply the previously obtained value by two.}

\subsection{Data from the 1000 genomes project}
\label{ss:1000}
In order to detect recent admixture in publicly available data, we downloaded 1000 Genomes data (phase 3) from \url{ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/}, as well as information on the sampling locations from \url{ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/integrated_call_samples_v3.20130502.ALL.panel} \cite{article}{Auton2015}. This is data from 661 individuals from Africa (AFR), 347 Admixed Americans (AMR), 504 East Asians (EAS), 503 Europeans (EUR) and 489 South-East Asians (SAS). The dataset comes with approximately 80 million SNPs. However, we use only a few of them known as the EUROFORGEN AIMset \cite{article}{Phillips2014} and Kidd AIMset \cite{article}{Kidd2014}, respectively. The former comes with 128 SNPs, and we ignore seven tri-allelic SNPs (rs17287498, rs2069945, rs2184030, rs433342, rs4540055, rs5030240, rs12402499), since our methods currently rely on bi-allelic SNPs. It was designed to distinguish Africa, Europe, East Asia, Native America, and Oceania, but was shown to perform well on the 1000 genomes dataset, also for distinguishing South Asia, even when ignoring the tri-allelic SNPs \cite{article}{Pfaffelhuber2019}.  The latter comes with 55 bi-allelic SNPs and was introduced as a global AIMset differentiating between 73 populations. We note that this AIMset is part of the Verogen MiSeq FGx™ Forensic Genomics Solution.
\\
The analysis of this dataset relies on allele frequencies used to estimate IA and PIA. Here, we use the samples of AFR, EAS, EUR and SAS. We did not use AMR since they are known to be admixed.

\subsection{Analysis of two collected individuals {\color{blue}with presumably recent admixture}}
\label{ss:AIMsself}
Within a larger study about biogeographical inference, buccal swabs from two individuals {\color{blue}were collected using a DNA-free swab (Sarstedt, Nümbrecht, Germany). The first individual reported one parent from Germany (Europe) and the other parent from the Philippines ((South-East)-Asian). The second individual reported a parent from Italy (Europe) and one from Venezuela (South America). }  Approval for collection and DNA analysis was obtained from the ethical committee of the University of Freiburg (414/18). DNA was extracted using the QIAamp Mini Kit (Qiagen, Hilden, Germany) and AIMs sequenced using the ForenSeq DNA Signature Prep Kit (Mix B) with the MiSeq FGx\textsuperscript{\textregistered} Reagent Micro Kit on a MiSeq FGx (all Verogen, San Diego, CA, USA). Sample preparation and sequencing was performed according to the Manufacturer's recommendations. SNPs were analyzed and exported for inclusion in the model using the
ForenSeq Universal Analysis Software (Verogen).

As a reference dataset for the analysis of the recent-admixture model (used for computing allele frequencies for continental populations), we use the Forensic {\em MPS AIMs Panel Reference Sets}, taken from \url{http://mathgene.usc.es/snipper/illumina_55.xlsx} which comes with the software {\sc Snipper} \cite{article}{Snipper2007}. This dataset contains data from the 1000 genomes project (504 out of 661 individuals from Africa (AFR) excluding the samples from African Caribbeans in Barbados and Americans of African Ancestry; 85 out of 347 Admixed Americans (AMR) only including Peruvians from Lima; 504 East Asians (EAS), 503 Europeans (EUR) and 489 South-East Asians (SAS)), as well as 13 Oceanian, Papua New Guinea, (OCE) samples from the Human Genome Diversity Panel. In the reference dataset, rs3811801 (contained in the ForenSeq DNA kit) is missing and therefore excluded from further analysis. This SNP has some discriminatory power for EAS (allele frequencies 1 (AFR); 1 (AMR); 0.49 (EAS); 0.99 (SAS)), as seen from the 1000 genomes data. Since data for rs1919550 and rs2024566 is missing for the Oceanic samples of the reference database, we also excluded these AIMs. Both only have low discriminatory power on a continental level. In total, this amounts to a total of 53 AIMs in the analysis, all of which are contained in the Kidd AIMset \cite{article}{Kidd2014}. Allele frequencies are displayed in Figure \ref{Sfig:allFreqs}.


\section{Results}

\subsection{{\color{blue} Analysis of a three-island model}}
We simulated genome-wide data from a sample, taken from a population genetic model with three islands, $A$, $B$ and $C$, using \cite{article}{Kelleher2016}. Migration is such that only $A, B$ and $B, C$ are connected, but not $A, C$; see Figure~\ref{fig:sim1}(A). We used a migration rate of 10 diploid individuals per connected islands (in both directions) per generation. More precisely, we simulate a sample of 400 individuals per island{\color{blue}, taken from a larger total population within each island}, each with 20~recombining chromosomes, each with about 2.5$\cdot 10^4$ SNPs. From these $\sim 5\cdot 10^5$ SNPs, {\color{blue} representing part of the human genome,} we use the step-wise approach from \cite{article}{Pfaffelhuber2019} to look for~10 Ancestry Informative Markers (AIMs). When using a naive Bayes approach as in {\sc snipper} \cite{article}{Snipper2007}, this AIMset gives a vanishing misclassification error for the task of classifying the 3$\times$400 simulated, non-admixed individuals. However, {\color{blue} for recently (first generation; see Section~\ref{S:insilico}) $A\times C$-admixed individuals, the same }  classifier fails in many cases and gives population $B$ as the best guess; see Figure \ref{fig:sim1}(B).

%\subsection{The recent-admixture model improves accuracy of ancestry proportions}
Subsequently, we used the admixture and recent-admixture model to estimate IA and PIA for both, non-admixed and first generation admixed individuals. We observe that the admixture model fails to give accurate estimates for IA in $A\times C$-recently-admixed individuals for two reasons. First, it correctly predicts that the individual is $A\times C$-admixed, but overestimates one of the two ancestral proportions. Second, and more severely, it confounds the signal for recent admixture with an ancestral proportion from island $B$. Figure~\ref{fig:sim1}(C) shows an example of an $A\times C$-recently-admixed individual, with misleading estimate for IA, but a more enlightening estimate for PIA. Here, IA predicts  ancestry mostly in $B$ when using admixture, since allele frequencies in $B$ are between $A$ and $C$.  However, recent-admixture correctly predicts two parents of different ancestry, one mostly $A$, the other mostly $C$. The individual is taken from all $A\times C$-admixed individuals, which are displayed in Figure~\ref{Sfig:sim2}.

To get a picture of all non-admixed and recently admixed samples, we computed errors for estimating IA as given in \eqref{eq:error} for the admixture and recent-admixture model. As described in the MM section, we average estimates $\hat q^M$ and $\hat q^P$ from the recent-admixture model, in order to compare to the true IAs. Figure~\ref{Sfig:sim3} displays these errors in all cases including non-admixed individuals, and all three cases of recent-admixture. Interestingly, binomial tests with the alternative that the recent-admixture model gives smaller errors show significant results in all but one case ($A, A\times B, A\times C, B\times C$: $p<0.001$, $B: p = 0.04$, $C: p=0.32$). This shows that on average the recent-admixture model performs better than the admixture model, even on non-admixed samples. 

\begin{figure}
  \parbox[t]{0.2\textwidth}{  
    \hspace{1cm} (A)

  ~
  
  \begin{tikzpicture}[every node/.style={circle}]
    \node[fill = blue, inner sep=.01cm] (n1) at (1,1)   {\color{white}$\;A\;\,$};
    \node[fill = green, inner sep=.01cm] (n2) at (2,2) {$\;B\;\,$};
    \node[fill = yellow, inner sep=.01cm] (n3) at (3,1) {$\;C\;\,$};
    \foreach \from/\to in
    {n1/n2,n2/n3}
    \draw[<->] (\from) -- (\to);
  \end{tikzpicture}
  }   \parbox[t]{0.32\textwidth}{  
    \hspace{2cm}(B)

  \includegraphics[width=0.3\textwidth]{abc_baryplot2.pdf}}
\parbox[t]{0.42\textwidth}{  
    \hspace{2.8cm} (C)

  ~

  % seed 210, mixed A x C 202
  \begin{tabular}{|l|r|}\hline
    \rule[-3ex]{0cm}{7ex} true \hfill \parbox{1cm}{\hfill $q$ \\ $q^M, q^P$}
    & \parbox{2cm}{\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (1cm, .4cm);
      \draw[fill=green] (1cm, 0cm) rectangle (1cm, .4cm);
      \draw[fill=yellow] (1cm, 0cm) rectangle (2cm, .4cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
    \end{tikzpicture}
    \\[-1ex]
      \begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=green] (2cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    \\[-2ex]\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
      \draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
      \draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    }
    \\ \hline\hline
    \rule[-2ex]{0cm}{5ex} admixture \hfill $\hat q$ 
    &  \parbox{2cm}{\begin{tikzpicture} 
\draw[fill=blue] (0cm, 0cm) rectangle (0.58cm, .4cm);
\draw[fill=green] (0.58cm, 0cm) rectangle (1.202cm, .4cm);
\draw[fill=yellow] (1.202cm, 0cm) rectangle (2cm, .4cm);
\draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
\end{tikzpicture}}
    \\ \hline
    \rule[-3ex]{0cm}{7ex} \parbox{1.8cm}{recent-\\admixture}
    \hfill $\hat q^M, \hat q^P$
    & \parbox{2cm}{
    \begin{tikzpicture} 
\draw[fill=blue] (0cm, 0cm) rectangle (1.704cm, .2cm);
\draw[fill=green] (1.704cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
\\[-2ex]\begin{tikzpicture} 
\draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
    }\\ \hline
  \end{tabular}

  }   \caption{(A) Illustration of the population model for the simulations. (B) Classification results for non-admixed and $A\times C$ admixed individuals (individuals from $A$: blue; individuals from $C$: yellow; $A\times C$ admixed individuals: grey). The individual from (C) is indicated by the arrow and is assigned to $B$ with probability almost 100\%. (C) For the individual from (B), estimates of IA ($\hat q$) using the admixture model and of PIA ($\hat q^M, \hat q^P$) using the recent-admixture model.}
  \label{fig:sim1}
\end{figure}


 \subsection{The recent-admixture model improves estimation accuracy of ancestry proportions for 1000 genomes project data}
For comparing the accuracy of the admixture and recent-admixture model, we extended our analysis of the errors for estimating IA to the 1000 genomes dataset. We excluded all Admixed Americans (AMRs) since they are known to have an admixed background \cite{article}{Eduardoff2016, Pfaffelhuber2019} and do not form a well-defined own group. As the true IA, we use the continental origins as described in the dataset (AFR, EAS, EUR and SAS). This means e.g.\ that we set $q^{\text{\sc true}}_{\text{EUR}}=1$ for a European sample in the dataset.

We ran three kinds of analyses. First, on the non-admixed samples. Second, we produced in silico recently admixed individuals with parents from the non-admixed samples (denoted AFR$\times$EUR etc.) and ran the analysis on these samples. Third, the analysis was performed on second-generation admixed samples, i.e.\ grandparents were taken from the non-admixed samples (denoted AFR/EAS$\times$EUR/SAS etc). In the first case, Figure~\ref{fig:nonmixed_EUROFORGEN} shows that the resulting errors for the admixture and recent-admixture model are almost identical, when using the EUROFORGEN AIMset. Overall, recent-admixture has a smaller error in 1364 out of 2157 cases, i.e.\ the hypothesis that the error for recent-admixture is at least as large as for admixture can be rejected (binomial test, $p<0.001$). % $p<10^{-34}$).
In the second case, Figure~\ref{fig:mixed_cases_EUROFORGEN}(A) shows clearly that errors for recent-admixture are smaller for all pairs of continents, when using the EUROFORGEN AIMset. More precisely, in 2279 out of 3000 individuals, recent-admixture is more accurate ($p<0.001$). % $p<10^{-186}$).
Third, for second-generation admixed individuals, Figure~\ref{fig:mixed_cases_EUROFORGEN}(B) displays errors in the cases (A)--(G) -- recall from Section~\ref{S:insilico} -- and shows that again, recent-admixture is more accurate, when using the EUROFORGEN AIMset. Here, recent-admixture outperforms admixture in 15761 out of 27500 cases (resulting in $p<0.001$) % $ p < 10^{-130}$).
A full list of~55 cases is displayed in Figure~\ref{fig:mixed_allcases_EUROFORGEN} in the SI. The corresponding results for the Kidd AIMset are similar and also found in the SI. We stress that the recent-admixture model not only gives significantly better estimates for IA, but also provides more information than the admixture model, since the genetic decomposition of both parents is estimated.

\begin{figure*}[htb]
  \hspace{3cm} (A) \hspace{8cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_EUROFORGEN.pdf}\vspace{2cm}}
    \hspace{1cm}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_cases_EUROFORGEN.pdf}}
  \end{center}
  \caption{\label{fig:mixed_cases_EUROFORGEN} For all first generation admixed samples (A) and second generation admixed samples (B), we computed IA from the admixture ($\hat q$) and recent-admixture ($\tfrac 12 (\hat q^M + \hat q^P)$) model using the EUROFORGEN AIMset. The distance to the true IA is computed as in \eqref{eq:error}. The cases in (B) are as described in Section~\ref{S:insilico}. }
\end{figure*}

\subsection{Power of the Likelihood-ratio test for recent admixture}
When fixing the minimal $\Delta\ell$  for deciding if a sample is recently admixed in the likelihood-ratio test for recent admixture (as described in MM), we obtain the power of the test for all cases of recent admixture. {\color{blue}For each case, we aim at distinguishing recent admixture with $(q^M, q^P)$ from ancient admixture with $q = (q^M+q^P)/2$. Displaying the false positives (i.e.\ positively tested anciently admixed) against true positives (i.e.\ positively tested recently admixed) in all cases for all possible values of $\Delta\ell$, we obtain the Receiver-Operation-Characteristic (ROC) curve \cite{article}{Fawcett2006}. }
%The optimal curve nearly hits~0 false positives with~100\% true positives and has an AUC (Area Under the Curve) of~1. 
As we see in Figure~\ref{fig:ROC_EUROFORGEN}, for the EUROFORGEN AIMset, the power of the test differs with the kind of {\color{blue} recent} admixture.  {\color{blue}The power is highest for first generation admixed (case (B)), and drops for one non-admixed parent (case (E)) and all grandparents from different continents (case (G)). It drops further if only half of the genome has two different ancestries (cases (D) and (F)). If the individual is not recently-admixed in first generation, but both parents are (case (C); note that this case is not recent-admixture, since $q^M = q^P$ holds), or it is non-admixed (case (A)), recent admixture does not hold and the test has no power to detect these cases.} The picture is nearly identical for the Kidd AIMset; see Figure~\ref{fig:ROC_Kidd}.

\begin{figure*}[htb]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{roc-curve-EUROFORGEN.pdf}
  \end{center}
      \caption{Using the EUROFORGEN AIMset, the ROC curve displays false positives (i.e.\ {\color{blue} anciently admixed with the same $q$,} against true positives (all cases of admixture in second generation)). In Figure~\ref{fig:ROC_EUROFORGEN} in the SI, a similar figure using the Kidd AIMset is given.}
  \label{fig:ROC_EUROFORGEN}
\end{figure*}

{\color{blue}
\subsection{Simulating $p$-values}
\label{ss:pvalues}We checked if our approach of using simulations to study $p$-values is conservative. Both, the report of a $p$-value of 1 if $|q^M - q^P| \leq 0.1$, and doubling the frequency of $\Delta\ell$-values above the $\Delta\ell$-value of the sample as reported $p$-value, are ad hoc. In order to analyse their performance, we use allele frequencies from the 1000 genomes dataset, again leave out AMR-individuals, and run simulations for individuals according to $H_0$. So, for fixed $q$, we obtain anciently admixed genomes as described in Section~\ref{S:insilico}, and run the computation of $p$-values. We investigate three cases, displayed in Figure~\ref{fig:pvalues} in the SI, for the following three cases: (i) non-admixed individuals, where $q_k=1$ for some $k\in\{\text{AFR}, \text{EAS}, \text{EUR}, \text{SAS}\}$; (ii) anciently admixed individuals from two populations, where $q_k q_{k'} = 0.5$ for some $k, k'$, denoted AFR+EAS,...; (iii) $q$ which is Dirichlet(1,...,1)-distributed (the uniform distribution on the simplex), denoted mixed. We see that $p$-values have a slight tendency to be not conservative only for samples mixed from all continents. All other cases, in particular, relevant cases of non-admixed individuals, have conservative $p$-values.}

{\color{blue}
\subsection{Structure as a confounding factor}
In order to test if hidden structure in the reference populations can have confounding consequences for the LR test, we simulated a scenario with a strongly structured reference population. Therefore, we merged EAS and SAS in the 1000 genomes dataset into one reference population, but simulated non-admixed EAS, SAS individuals as well as other cases and applied the LR test. The simulated individuals from either EAS or SAS showed an increase in homozygous sites compared to the Hardy-Weinberg expectation from the structured reference. Therefore, hidden substructure in the reference populations causes the opposite effect than recent admixture would, which is an increase in heterozygous sites.  Consequently, this scenario leads to conservative $p$-values, as can be seen from Figure \ref{fig:hiddenstructure} in the SI.}

\subsection{The LR test identifies recent admixture in the 1000 genomes dataset}
From the 1000 genomes dataset, we highlight individuals which give significant results for the test of recent admixture for the EUROFORGEN and Kidd AIMsets. As trainingset, for estimating allele frequencies, we use the individuals from \url{http://mathgene.usc.es/snipper/illumina_55.xlsx} which are part of the 1000 genomes dataset; see Section \ref{ss:AIMsself}. 
%(Recall that 85 Admixed Americans are included here, but two African populations are excluded.)
In Figure~\ref{fig:NA20278EUROFORGEN}, we give the result of the most extreme individual (in the sense of the {\color{blue} smallest $p$-value for both AIMsets} observed in the whole sample), a male from the African American (ASW) population. We note that it is known that the ASW population is admixed \cite{article}{Eduardoff2016}, but until now, it has not been tested if admixture is recent. In  Section S2.5 in the SI, we analyse all individuals from the dataset with a $p$-value below 5\% for both, the EUROFORGEN and Kidd AIMset. In total, we find eight such individuals, four with African origin (all ASW) and four admixed Americans (two CLM, Colombian ancestry; one MXL, Mexican ancestry; one PUR, Puerto Rician ancestry). In  Section S2.5 in the SI, we see that heterozygous sites are in fact the drivers of the large difference in log-likelihood between the recent-admixture and the admixture model. 

%The likelihood-ratio test indicates that it is $e^{6.747} \approx 8.5 \cdot 10^2$ times more likely that the individual is recently admixed than non-recently, when using the EUROFORGEN AIMset and $e^{10.010} \approx 2.2 \cdot 10^4$ times more likely for the Kidd AIMset. Similar results appear in the Admixed American population, where we find an individual which appears to be recently admixed from Europe and Admixed American (individual NA19720); see Section S2.5 in the SI. 
%We note, however, that the result for recent admixture may in some cases depend on the AIMset used. E.g., for another Admixed American from Mexico in the sample, NA19719, Figures~\ref{fig:NA19720EUROFORGENE} and \ref{fig:NA19720Kidd} show that the evidence for recent admixture using the EUROFORGEN AIMset is much greater that for the Kidd AIMset.

\begin{figure*}[!htb]
  (A) \hspace{4cm} (B)
  
  \parbox{4.5cm}{
\begin{tikzpicture}[scale=1.5] 
\draw[fill=red] (0cm, 0cm) rectangle (0.882cm, .4cm);
\draw[fill=white] (0.882cm, 0cm) rectangle (0.924cm, .4cm);
\draw[fill=blue] (0.924cm, 0cm) rectangle (0.924cm, .4cm);
\draw[fill=green] (0.924cm, 0cm) rectangle (2cm, .4cm);
\draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .4cm);
\end{tikzpicture}}
  \parbox{4.5cm}{
  \begin{tikzpicture}[scale=1.5] 
\draw[fill=red] (0cm, 0cm) rectangle (1.822cm, .2cm);
\draw[fill=white] (1.822cm, 0cm) rectangle (1.944cm, .2cm);
\draw[fill=blue] (1.944cm, 0cm) rectangle (1.998cm, .2cm);
\draw[fill=green] (1.998cm, 0cm) rectangle (1.998cm, .2cm);
\draw[fill=yellow] (1.998cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
\\[-1.3ex]\begin{tikzpicture}[scale=1.5]
\draw[fill=red] (0cm, 0cm) rectangle (0.026cm, .2cm);
\draw[fill=white] (0.026cm, 0cm) rectangle (0.026cm, .2cm);
\draw[fill=blue] (0.026cm, 0cm) rectangle (0.026cm, .2cm);
\draw[fill=green] (0.026cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
}\\[2ex]
\parbox[t]{10cm}{AFR$\!$ \begin{tikzpicture}[baseline=(current bounding box.south)]\draw [fill=red] (0,0) rectangle
      (.8cm,0.25cm);\end{tikzpicture}, %\\ \noindent 
    AMR$\!$ \begin{tikzpicture}[baseline=(current bounding
      box.south)]\draw [fill=white, draw=black] (0,0) rectangle
      (.8cm,0.25cm);\end{tikzpicture}, %\\ \noindent
    EAS$\!$ \begin{tikzpicture}[baseline=(current bounding
      box.south)]\draw [fill=blue] (0,0) rectangle
      (.8cm,0.25cm);\end{tikzpicture},     EUR$\!$ \begin{tikzpicture}[baseline=(current bounding
      box.south)]\draw [fill=green] (0,0) rectangle
      (.8cm,0.25cm);\end{tikzpicture} 
      %,     SAS$\!$ \begin{tikzpicture}[baseline=(current bounding box.south)]\draw [fill=yellow] (0,0) rectangle      (.8cm,0.25cm); \end{tikzpicture}
  }
  
  \vspace{0.5cm}
  
  (C) \hspace{3cm}

%\begin{tabular}{|l|r|r|}\hline
%  & number & \parbox{2.3cm}{\hfill contribution \vphantom{$\int$}\\[-1ex] \vphantom{$\int$}\hfill to $\Delta\ell$} \\\hline\hline
%  heterozygote sites & 51 & 15.177 \\\hline
%  homozygote sites & 70 & -8.430 \\\hline\hline
%  sum & 121 & 6.747 \\ \hline
%\end{tabular}
{\color{blue}
\begin{tabular}{|l|r|}\hline
$\Delta\ell$ & 6.747 \\ \hline
$p$-value & $<0.001$ \\ \hline
\end{tabular}}

  \caption{\label{fig:NA20278EUROFORGEN} Analysis of NA20278 from the 1000 genomes dataset using the EUROFORGEN AIMset. (A) Estimated IA. (B) Estimated PIA. (C) {\color{blue}$p$-value found by simulation, and} difference in log-likelihood $\Delta\ell$ from the likelihood-ratio test. More details on these contributions, broken down to single AIMs, is found in Section S2.5 in the SI.}
\end{figure*}

\subsection{Individuals with {\color{blue}presumably} recent admixture sequenced with the Verogen MiSeq FGx™ Forensic Genomics Solution}
{\color{blue}To further explore the performance of our method in an applied setting and to compare it to other current methods, we analyzed data from two individuals, generated with the Verogen MiSeq FGx™ Forensic Genomics Solution.} For the German/Philippine female, when using a classification tool based on a naive Bayes approach (e.g.\ {\sc snipper}), data from the 53 autosomal markers indicated a 61\% chance to be European and 39\% to be South-East-Asian (with reference samples from India, Pakistan etc). The ForenSeq Universal Analysis Software did not provide a clear classification result into one cluster of the training dataset, but the sample falls together with the Admixed American samples of the 1000 Genome project. The closest cluster is mainly comprised of samples from the 1000 genome populations from Puerto Rico and Colombia. The use of the admixture model leads to a mixed ancestry from Europe, East-Asian and Oceania; see Figure~\ref{fig:ind83}(A). Using the recent-admixture model, one parent with European and one parent with mainly an East-Asian ancestry are revealed which fits the self-declaration Figure~\ref{fig:ind83}(B). This individual has a $p$-value of $0.001$, such that the recent-admixed model is favoured Figure~\ref{fig:ind83}(C).

For the Italian/Venezuelan male, when using a naive Bayes classifier, his DNA is classified as European. The ForenSeq Universal Analysis Software provides a classification rather into the European cluster, but states the closest centroid in which also single reference samples from the 1000 genome project from European as well as Middle- and South-American ancestry fall into. The admixture model estimates mainly European ancestry, and contribution of South-East-Asian, and a small fraction African ancestry Figure~\ref{fig:ind83}(D). The recent-admixture model estimates one parent with European ancestry (explained by the Italian father) and one parent with mostly South-East Asian origin Figure~\ref{fig:ind83}(E). {\color{blue}However, the corresponding $p$-value shows that this recent-admixture is not significant Figure~\ref{fig:ind83}(F). }

\begin{figure*}[htb]
  (A) \hspace{8.5cm} (D)
  
  \parbox{9cm}{
  \begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0cm, .4cm);
\draw[fill=white] (0cm, 0cm) rectangle (0cm, .4cm);
\draw[fill=blue] (0cm, 0cm) rectangle (0.482cm, .4cm);
\draw[fill=green] (0.482cm, 0cm) rectangle (1.716cm, .4cm);
\draw[fill=gray] (1.716cm, 0cm) rectangle (1.956cm, .4cm);
\draw[fill=yellow] (1.956cm, 0cm) rectangle (2cm, .4cm);
\end{tikzpicture}
}
  \parbox{5cm}{
  \begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0.144cm, .4cm);
\draw[fill=white] (0.144cm, 0cm) rectangle (0.27cm, .4cm);
\draw[fill=blue] (0.27cm, 0cm) rectangle (0.27cm, .4cm);
\draw[fill=green] (0.27cm, 0cm) rectangle (1.698cm, .4cm);
\draw[fill=gray] (1.698cm, 0cm) rectangle (1.698cm, .4cm);
\draw[fill=yellow] (1.698cm, 0cm) rectangle (2cm, .4cm);
\end{tikzpicture}} \hspace{1cm}

  \vspace{0cm}
  
  (B) \hspace{8.5cm} (E) \hspace{4cm}

  \parbox{9cm}{
  \begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=white] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=blue] (0cm, 0cm) rectangle (0.99cm, .2cm);
\draw[fill=green] (0.99cm, 0cm) rectangle (0.99cm, .2cm);
\draw[fill=gray] (0.99cm, 0cm) rectangle (1.402cm, .2cm);
\draw[fill=yellow] (1.402cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
\\[-2ex]\begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=white] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=green] (0cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=gray] (2cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}}
  \parbox{5cm}{\begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0.266cm, .2cm);
\draw[fill=white] (0.266cm, 0cm) rectangle (0.504cm, .2cm);
\draw[fill=blue] (0.504cm, 0cm) rectangle (0.504cm, .2cm);
\draw[fill=green] (0.504cm, 0cm) rectangle (0.782cm, .2cm);
\draw[fill=gray] (0.782cm, 0cm) rectangle (0.782cm, .2cm);
\draw[fill=yellow] (0.782cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
\\[-2ex]\begin{tikzpicture} 
\draw[fill=red] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=white] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
\draw[fill=green] (0cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=gray] (2cm, 0cm) rectangle (2cm, .2cm);
\draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
\end{tikzpicture}
  }

~

  \parbox[t]{15cm}{ \noindent Colors are AFR \crule[red]{.5cm}{.3cm}, AMR \fbox{\crule[white]{.5cm}{.3cm}}, EAS \crule[blue]{.5cm}{.3cm}, EUR \crule[green]{.5cm}{.3cm}, OCE \crule[gray]{.5cm}{.3cm}, SAS \crule[yellow]{.5cm}{.3cm}.
}
  

  \vspace{.5cm}

  (C) \hspace{8.5cm} (F) \hspace{4cm}
  
  \vspace{.2cm}

\noindent    
 {\color{blue}
\begin{tabular}{|l|r|}\hline
$\Delta\ell$ & 3.513 \\ \hline
$p$-value & $0.001$ \\ \hline
\end{tabular}}
% \begin{tabular}{|l|r|r|}\hline
%  & number & \parbox{2.3cm}{contribution \vphantom{$\int$}\\[-1ex] \vphantom{$\int$}to $\Delta\ell$} \\\hline\hline
%  heterozygote sites & 19 & 4.639 \\\hline
%  homozygote sites & 34 & -1.127 \\\hline\hline
%  sum & 53 & 3.513 \\ \hline
%\end{tabular}
\hspace{6cm}
{\color{blue}
\begin{tabular}{|l|r|}\hline
$\Delta\ell$ & 0.820 \\ \hline
$p$-value & $0.229$ \\ \hline
\end{tabular}}

%\begin{tabular}{|l|r|r|}\hline
%  & number & \parbox{2.3cm}{contribution \vphantom{$\int$}\\[-1ex] \vphantom{$\int$}to $\Delta\ell$} \\\hline\hline
%  heterozygote sites & 18 & 1.766 \\\hline
%  homozygote sites & 35 & -0.946 \\\hline\hline
%  sum & 53 & 0.820 \\ \hline
%\end{tabular}

    
  \caption{\label{fig:ind83} (A)--(C) Same as Figure~\ref{fig:NA20278EUROFORGEN}, but for the first collected individual. (D)--(F) Same for the second collected individual. In Section S2.6 in the SI, contributions to $\Delta\ell$ are broken down to single AIMs.}
\end{figure*}



\subsection{Runtimes} 
The analysis of the admixture and recent-admixture model is fast. The main reason is that allele frequencies are only computed from a reference database (and not estimated on the fly, as in {\sc STRUCTURE} and {\sc ADMIXTURE}). As a consequence, runtimes scale linearly with the number of analysed traces. E.g.\ once allele frequencies for all AIMs from the reference dataset are given, {\color{blue} the estimation of IA and PIA} from one of the $500\cdot 55 = 27500$ individuals in Figure~\ref{fig:mixed_cases_EUROFORGEN}, and which are created in silico, takes about 1.5 seconds per individuals on a standard laptop computer using the statistical language~R. {\color{blue}The most time-consuming procedure, however, is the computation of $p$-values via simulation. Here, for each simulated individual, again IA and PIA have to be computed, which then takes about three minutes per sample, if 100 individuals are simulated.}

\section{Discussion}
Our first objective was to assess how recent admixture affects the outcome of standard methods of BGA inference. {\color{blue} Recall that BGA is inferred either using a (all-or-nothing) classifier (as in {\sc SNIPPER} \cite{article}{Snipper2007}), or by estimating individual ancestry in a mixed membership model (as in {\sc STRUCTURE} \cite{article}{Pritchard2000} and {\sc ADMIXTURE} \cite{article}{Alexander2009}).} Using simulated island populations as well as data generated with the Verogen MiSeq FGx™ Forensic Genomics Solution, we provide in depth evidence that using classification software such as {\sc snipper} (all-or-nothing classifiers) is not suitable in {\color{blue}recently admixed individuals}, as has been suggested by \cite{article}{Cheung2018}. Prestes et al. (2016) \cite{article}{Prestes2016} notes that mixed membership models as used in {\sc STRUCTURE} \cite{article}{Pritchard2000} and {\sc ADMIXTURE} \cite{article}{Alexander2009} are in many cases capable of inferring mixed ancestry in such individuals and \cite{article}{Cheung2018} introduced a genetic distance algorithm, which improves results from {\sc ADMIXTURE} in specific cases. However, we show that also the {\color{blue} more sophisticated methods} for BGA inference, {\sc STRUCTURE} and {\sc ADMIXTURE}, can be misleading in recently admixed individuals. These models fail because the assumption of equal chances for the two alleles at a locus to stem from any population is violated in recently admixed individuals. {\color{blue} Here, as well as for various cases without recent admixture, the recent-admixture model on average gives more accurate results. }

Our second objective was improving the inference of BGA (or estimation of admixture composition, IA) and to develop a test to identify recently admixed individuals. We used the excess of heterozygote sites in recently admixed individuals -- called the Wahlund principle \cite{article}{Wahlund1928} -- in order to estimate parental admixture (see also \cite{article}{Crouch2012}) and test for recent admixture. {\color{blue}The recent-admixture model we present here is an extension of the admixture model. Whereas the admixture model assumes that an individual comes with its individual admixture (IA) from all reference populations, the extension presented here lies in the assumption that both parents can have different IA, which we call parental individual admixture (PIA). In particular, the recent-admixture model contains the well-established admixture model (see e.g.\ \cite{article}{Pritchard2000, Alexander2009}) as a special case. We note that the model cannot be easily generalized by e.g.\ introducing grand-parent individual admixture, since the signal of recent admixture of parents is lost on the level of the trace individual, at least if we assume linkage equilibrium between markers.}
Note that we still assume that AIMs are spread out in the genome, leading to independent segregation. This is in contrast to approaches using genome-wide dense SNP data, where linkage has to be taken into account \cite{article}{Zhou2015, Pei2020}. Another assumption made in our analysis is that allele frequencies in all populations are provided in a reference database. {\color{blue}This is different from the approach taken in {\sc STRUCTURE}, where IA} and allele frequencies are estimated at the same time. {\color{blue}In addition, we stress that the total number $K$ of populations, which is a sensible parameter in various applications of {\sc STRUCTURE}, see e.g.\ \cite{article}{pmid20683484}, here comes as a property of the reference database, i.e.\ is given externally.} A consequence of the {\sc STRUCTURE} approach is that the analyzed traces will shift the allele frequencies in the clusters that are posthoc assumed to represent the reference populations. We believe that in forensic genetics, the outcome of the analysis should not depend on e.g.\ the number of traces which are studied, and thus a method that relies on a reference data base should be preferred. A welcome side effect of leveraging a reference data base is that our analysis is faster than {\sc STRUCTURE} and {\sc ADMIXTURE}. Since our method does not need genome-wide data, but only a few AIMs, analysis can be further accelerated and applied to forensic casework. 

In inference of BGA in general, {\color{blue} and for recent-admixture in particular}, several hypothesis can be tested by a likelihood ratio (LR) test. For example, Tvedebrink and colleagues have recently developed statistical tests with the null hypothesis that the trace is a non-admixed sample from one of the populations in the reference database \cite{article}{Tvedebrink2018}. This test was extended in \cite{article}{Tvedebrink2019} to the null hypothesis that the trace is a recent admixture of the (non-admixed) parents from two populations in the reference database. In contrast, the LR test presented here tests if the recent-admixture model (with two parents with different IA) fits the data significantly better than the admixture model (where both alleles at a locus have the same chance given by IA to originate in any of the continents). As expected for a method that relies on a genome-wide increase in heterozygosity to detect recently admixed individuals, we see that heterozygote sites are responsible for the biggest contribution to the increase in likelihood in the recent-admixture model. {\color{blue} Related to this Wahlund effect are thoughts on confounding factors for inference of recent-admixture. At least, we have shown that hidden structure within the reference database and the trace does not confound recent-admixture, since hidden structure leads to an excess of homozygotes (rather than an excess of heterozygotes). At least, only few mechanisms are known to inflate the number heterozygotes. As for heterozygote advantage, it is hardly conceivable that such a locally (in the genome) acting selective force confounds {\color{blue}genome-wide patterns of} recent admixture}

{\color{blue} For determining $p$-values and the specificity of the LR test, we rely on simulations. Thus, the asymptotic LR-theory does not apply and the distribution of the LR-test statistics depends on the true IA. On the way, we had to make some ad-hoc decision rules, one of them being that the estimated IA for the two parents has to be sufficiently different. Here, a better understanding of the theoretical distribution of the LR test statistics, would certainly be desirable. The sensitivity or power of our LR test to detect recent-admixture is highest  if the two parents have non-admixed IA from different populations. Other cases, such as four grand-parents coming from four different populations, unsurprisingly give a lower power. 
}

In our real-world samples, we detected signs of recent-admixture both in the 1000 genomes dataset and the {\color{blue} collected} individuals. Note that the 1000 genomes project specifically targeted individuals that are most likely to have all of their grandparents in only one location or population \cite{article}{1000Gconsentsheet}. Nevertheless, we find striking evidence for recent admixture in some African individuals from the southwest of America{\color{blue}, as well as for some admixed Americans}. For the two presumably admixed samples that were analyzed with ForenSeq, the analysis of recent-admixture was compared with the self-reporting of the individuals. The ancestry information is therefore based on the assumption of having a biologically correct family tree and correct information on birth place and ancestry. In case of the female individual, Philippine and German ancestry were declared for the last four generations. This individual was successfully identified as recently admixed. The other individual, with self-reported maternal Venezuelian ancestry of the last two generations, was ”insecure” about the generation before. {\color{blue}Both IA and PIA suggested a proportion of SAS ancestry for this individual, which seems misleading. However, it has been noted that admixed South American populations have a high genetic overlap with SAS populations, making it difficult to differentiate these populations even when analyzed with genome-wide, dense SNP data \cite{article}{pmid30845922}. This suggests that the AIM based approaches that we applied might not be able to resolve these differences and misidentify the South American ancestry as SAS ancestry in this sample. That we have no population from Venezuela in the reference database might have added to the apparent misassignment of both, IA and our newly developed PIA. Because Europeans contributed substantially to the genetic make up of the Venezuelan population via ancient admixture \cite{article}{pmid18027815}, our method might lack power to detect the recent admixture event in this case.}


%{\color{blue}Recently, some LR tests have been developed in the area of BGA inference. While the methods developed in \cite{article}{Tvedebrink2018} and \cite{article}{Tvedebrink2019} test if the trace is represented in the reference database, we developed a test for recent admixture. Combinations of such tests have the potential to obtain a clearer picture of the ancestry of individuals. In order to achieve this, methods must be able to rely on reference databases of high quality. First, this means that markers within the database have a high distinctive power for all populations in the database. Second, BGA results can be misleading if not all ancestral groups that are present in the trace are represented in the reference database. Factors that decrease the power of existing methods and lead to confounding results will be subject to future research.}

%Another explanation for the diverging reported ancestry from the estimation result on parental ancestry could be that the only admixed American population (Peruvians) in the reference dataset is genetically further apart from Venezuela than the South-East Asian reference samples.

%The recent-admixture model gives accurate results even on small AIMsets. Most importantly, we see in our simulations and real world examples, that the results from the recent-admixture model outperform the admixture model in virtually all cases. In addition, they provide additional insights of the recent ancestry of the trace which can guide casework. 





\subsubsection*{Acknowledgements}
We thank Denise Syndercombe Court for comments on an earlier version of the manuscript, {\color{blue} and two anonymous reviewers for several suggestions, in particular for pushing us to report $p$-values}. The sequencing results are part of a larger study which is partly funded by the Wissenschaftliche Gesellschaft Freiburg. 


\bibliographystyle{article}{unsrt}
\bibliography{article}{BGA}{References}
%\bibliographystyle{chicago} \bibliography{BGA}

\newpage
\setcounter{page}{1}
\setcounter{section}{0}
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{equation}{0}
\thispagestyle{empty}

\begin{center}
 \title{\LARGE Supporting Information: Inference
  of recent admixture using genotype data}

~~

\author{\sc Peter Pfaffelhuber, Elisabeth Sester-Huss, Franz Baumdicker, \\
  \sc Jana Naue, Sabine Lutz-Bonengel, Fabian Staubach}

~

\date{\today}

\maketitle
  
\end{center}

\renewcommand{\theequation}{S\arabic{equation}}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\thesection}{S\arabic{section}}


\section{Theory}
We write down the admixture model and derive a method to estimate
Individual Admixture (IA) in the case when allele frequencies within
populations are not updated. In addition, we give the recent-admixture
model, where an individual is allowed to have parents with different
admixture proportions. We take the following notation for the
reference database:
\begin{align*}
  K: & \text{ number of ancestral populations,}
  \\ M: & \text{ number of markers,}
  \\ p_{mk}: & \text{ frequency of allele 1 at (bi-allelic) marker $m$ in population $k$.}
\end{align*}
In addition, we consider one additional diploid genome (called the
trace) $(G_{m1}, G_{m2})_{m=1,...,M}$, or $(G_m)_{m=1,...,M}$ with
$G_m = G_{m1} + G_{m2}$ if phase is not known. The goal is to estimate
admixture proportions $(q_k)_{k=1,...,K}$ (or
$(q^M_{k})_{k=1,...,K}, (q^P_{k})_{k=1,...,K}$) of this additional
genome. Recall that we take the approach from
\cite{SI}{Chakraborty1986} and \cite{SI}{Hanis1986} and do
not update $p_{mk}$'s during the analysis.

\subsection{The admixture model}
In \cite{SI}{Alexander2009}, \cite{SI}{Pritchard2000},
\cite{SI}{Tang2005} and elsewhere, the main goal is to maximize the
log-likelihood (see also (1) and (2) of \cite{SI}{Tang2005})
\begin{align*}
  \ell(q|G) = \sum_{m=1}^M \sum_{a=1,2} \log\Big(\sum_{k=1}^K \alpha_{mk}^{\color{blue}a}q_k\Big),  
\end{align*}
where
$$\alpha_{mk}^{\color{blue}a} := \begin{cases} p_{mk}, & \text{ if }G_{ma}=1,\\ 1-p_{mk}, & \text{ if }G_{ma}=0.\end{cases}$$
is the frequency of the observed allele in copy $a$ of marker $m$ in
population $k$. (Note that $e^{\ell(q|G)}$ is the probabiltiy of
observing $(G_{ma})_{m=1,...,M; a=1,2}$, if every allele is picked
independently from population $k$ with probability $q_k$.) Assuming
that phase is not known, and with
\begin{align}
  \label{eq:amkl}
  \alpha_{mkl}:= \alpha_{mk}^{\color{blue}1}\alpha_{ml}^{\color{blue}2} = \begin{cases}
    p_{mk}p_{ml}, & \text{ if } G_{m1} + G_{m2} = 2, \\
    p_{mk}(1-p_{ml}) + (1-p_{mk})p_{ml}, & \text{ if } G_{m1} + G_{m2} = 1, \\
    (1-p_{mk})(1-p_{ml}), & \text{ if } G_{m1} + G_{m2} =
    0, \end{cases}
\end{align}
note that the log-likelihood can as well be written as
\begin{align*}
  \ell(q|G) = \sum_{m=1}^M \log\Big(\sum_{k,l=1}^K \alpha_{mkl}q_kq_l\Big).
\end{align*}
We set $\beta_m(q) := \sum_{k=1}^K p_{mk}q_k$ and analyse the last sum
by distinguishing the case $G_m=2$, where
\begin{align*}
  \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = \Big(\sum_{k=1}^kp_{mk}q_k\Big)^2 = \beta_m(q)^2
    \intertext{while for $G_m=1$ we find}
    \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = 2\Big(\sum_{k=1}^kp_{mk}q_k\Big)\Big(1-\sum_{k=1}^kp_{mk}q_k\Big) = 2\beta_m(q)(1-\beta_m(q))
    \intertext{and for $G_m=0$}
    \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = \Big(1-\sum_{k=1}^kp_{mk}q_k\Big)^2 = (1-\beta_m(q))^2,
\end{align*}
such that
\begin{align}  \label{eqSI:logLadmixture}
  \ell(q|G) = \sum_{m=1}^M \log\Big(\binom{2}{G_m} \beta_m(q)^{G_m}(1-\beta_m(q))^{2-G_m}\Big)
\end{align}

\begin{lemma}\label{l1}
  The maximum of $q\mapsto \ell(q|G)$ under the constraint
  $\sum_{k=1}^K q_k = 1$ solves
  \begin{align}\label{eq:0}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K
    \alpha_{mk'l'}q_{k'}q_{l'}} = 1, \qquad k=1,...,K.
  \end{align}
\end{lemma}

\begin{remark}
  \begin{enumerate}
  \item Assume we add a set of {\em Ancestry Uninformative Markers},
    i.e.\ a set of markers $\tilde m = 1,...,\tilde M$, with
    frequencies not depending on the population, i.e.\
    $\alpha_{\tilde m k l} = \alpha_{\tilde m k'l'}$ for
    $k,l,k,l''=1,...,K$. For these markers,
    $$\sum_{\tilde m = 1}^{\tilde M} \sum_{l=1}^K \frac{\alpha_{\tilde      mkl}q_l}{\sum_{k',l'=1}^K \alpha_{\tilde mk'l'}q_{k'}q_{l'}} =
    \tilde M.$$ This implies that $q$ is a solution of \eqref{eq:0}
    without these markers iff $q$ is a solution of \eqref{eq:0} if
    these markers are included. This might be reassuring.
  \item Let us have a closer look at the left hand side of
    \eqref{eq:0}. We set $\beta_m := \sum_{k=1}^K p_{mk} q_k$. For
    $G_m=2$,
    \begin{align*}
      \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & = 
        \sum_{l=1}^K \frac{p_{mk}p_{ml}q_l}{\sum_{k',l'}p_{mk'}p_{ml'}q_{k'}q_{l'}} = \frac{p_{mk}}{\beta_m}.
        \intertext{For $G_m=1$, we have}
        \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & =
        \sum_{l=1}^K \frac{(p_{mk}(1-p_{ml}) + (1-p_{mk})p_{ml})q_l}{\sum_{k',l'}(p_{mk'}(1-p_{ml'})
        + (1-p_{mk'})p_{ml'})q_{k'}q_{l'}}
      \\ & = \frac{p_{mk}(1-\beta_m) + (1-p_{mk})\beta_m}{2\beta_m(1-\beta_m)}
           = \frac 12\Big(\frac{p_{mk}}{\beta_m} + \frac{1-p_{mk}}{1-\beta_m}\Big)
           \intertext{and for $G_m=0$}
           \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & = 
        \sum_{l=1}^K \frac{(1-p_{mk})(1-p_{ml})q_l}{\sum_{k',l'}(1-p_{mk'})(1-p_{ml'})q_{k'}q_{l'}} = \frac{1-p_{mk}}{1-\beta_m}.
    \end{align*}
    In total, this gives that $q$ needs to solve
    \begin{align*}
      \frac{1}{2M} \sum_{m=1}^M \Big(G_m \frac{p_{mk}}{\beta_m} + (2-G_m)\frac{1-p_{mk}}{1-\beta_m}\Big) = 1.
    \end{align*}
    In order to find a solution, we reformulate as the fixed point
    equation
    \begin{align}\label{eqSI:fixed}
      q_k = f_k(q) \text{ for }f_k(q) =
      \frac{1}{2M} \sum_{m=1}^M \Big(G_m \frac{p_{mk}}{\beta_m} + (2-G_m)\frac{1-p_{mk}}{1-\beta_m}\Big)q_k,
      \qquad k =1,...,K.
    \end{align}
This can be done numerically by iterating
$q_{n+1} = (f_k(q_n))_{k=1,...,K}$ until convergence. ({\color{blue} Note, however, that we do not have a formal proof of convergence for this iteration. At least, in our numerical experiments, convergence  -- in the sense that we continue the iteration until $|q_{n+1} - q_n|< 10^{-6}$ -- always happened.}) We note that this approach is essentially the same as in the EM-algorithm from \cite{SI}{Tang2005}, but combining the expectation and maximization steps, since we do not update allele frequencies. In addition, although maximizing \eqref{eqSI:logLadmixture} could also be handled using a Newton method as in \cite{SI}{Alexander2009}, this approach has the advantage that $q_n$'s are positive in all steps, and the sum of all entries in $q_n$ is always~1. Moreover, the iteration is computationally fast if only a small or moderate number of {\color{blue} markers} is considered.

  \end{enumerate}
\end{remark}

\begin{proof}[Proof of Lemma~\ref{l1}]
  We use the theory of Lagrange multipliers, since we need to maximize
  $\ell$ over $q$ under the constraint $\sum_k q_k=1$.  Since
  \begin{align*}
    \frac{\partial \ell(q|G)}{\partial q_k} & = \sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K
                                              \alpha_{mk'l'}q_{k'}q_{l'}},
  \end{align*}
  we have to solve the system of equations
  \begin{align*}
    \lambda & = \sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}}, \qquad k=1,...,K,
    \\
    1 & = \sum_{k=1}^K q_k.
  \end{align*}
  It is easy to eliminate $\lambda$, since the last equation gives
  \begin{align*}
    \lambda & = \lambda \sum_{k=1}^K q_k = \sum_{m=1}^M \sum_{k,l=1}^K
              \frac{\alpha_{mkl}q_kq_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}} = M.
  \end{align*}
  So, we are left with finding $q = (q_k)_{k=1,...,K}$ such that
  \begin{align*}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}} = 1, \qquad k=1,...,K.
  \end{align*}
\end{proof}


\subsection{The recent-admixture model}
\noindent
For the recent-admixture-version, we want to estimate
$q^M=(q^M_{k})_{k=1,...,K}, q^P=(q^P_{k})_{k=1,...,K}$, where
$q^M_{k}$ and $q^P_k$ are the fractions of the genomes of the mother
and father, respectively, which come from population $k$.  This
assumption implies that the log-likelihood is
$$ \ell(q^M, q^P|G) = \sum_{m=1}^M  \log\Big(\sum_{k,l=1}^K \alpha_{mkl}q_{k}^M q_l^P\Big),$$
where $\alpha_{mkl}$ is given as in \eqref{eq:amkl}.  As is apparent
from the log-Likelihood function, the recent-admixture model
generalizes the admixture model. Put differently, choosing
$q^M = q^P = q$ in the recent-admixture model gives the admixture
model. We note that again, the log-likelihood can be written
differently,
\begin{equation}
  \begin{aligned}\label{eqSI:logLrecentadmixture}
    \ell(q^M, q^P|G) & = \sum_{m=1}^M  \log\Big( 1_{G_m=2} \beta_m(q^M) \beta_m(q^P)
    \\ & \qquad \qquad \qquad + 1_{G_m=1}(\beta_m(q^M) (1-\beta_m(q^P))
    + (1-\beta_m(q^M)) \beta_m(q^P))
    \\ & \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad + 1_{G_m=0}(1-\beta_m(q^M))(1-\beta_m(q^P))\Big).    
  \end{aligned}
\end{equation}

\begin{lemma}\label{l2}
  The maximum of $(q^M, q^P)\mapsto \ell(q^M, q^P,G)$ under the
  constraint $\sum_{k=1}^K q_k^M = \sum_{k=1}^K q_k^P = 1$ solves
  \begin{align}\label{eq:0recent}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
    = \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K\frac{\alpha_{mkl}q_l^P}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^P_{k'}q^M_{l'}} = 1, \qquad k=1,...,K.
  \end{align}
\end{lemma}

\begin{remark}
  \begin{enumerate}
  \item Note that \eqref{eq:0recent} is symmetric in $q^M$ and $q^P$,
    i.e.\ if $(q^M, q^P)$ solve \eqref{eq:0recent}, another solution
    is given by $(q^P, q^M)$.
  \item Again, we can turn \eqref{eq:0recent} into fixed point
    equations. To derive it, we again have a closer look at the left
    hand side of \eqref{eq:0recent}. For
    $\beta_m(q) := \sum_k p_{mk} q_k$, we have for $G_m=2$
    \begin{align*}
      \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{p_{mk} \beta_m(q^M)}{\beta_m(q^M)\beta_m(q^P)} = \frac{p_{mk}}{\beta_m(q^P)},
        \intertext{for $G_m=1$}
        \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{p_{mk} (1-\beta_m(q^M)) + (1-p_{mk})\beta_m(q^M)}{\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P)}
        \intertext{and for $G_m=0$}
        \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{(1-p_{mk})(1- \beta_m(q^M))}{(1-\beta_m(q^M))(1-\beta_m(q^P))} = \frac{1-p_{mk}}{1-\beta_m(q^P)}.
    \end{align*}
    So, we suggest to iteratively compute
    \begin{align}\label{eqSI:fixedRecent}
      \tilde q^P & = f(q^M, q^P) \text{ and } \tilde q^M = f(q^P, q^M)
                   \intertext{for $f(q, q') = (f_k(q, q'))_{k=1,...,K}$ with} \notag
                   f_k(q, q') & := \frac{1}{M}\sum_{m=1}^M \Big(1_{G_m=2} \frac{p_{mk}}{\beta_m(q')}
      \\ \notag & \qquad \qquad \qquad + 1_{G_m=1} \frac{(p_{mk} (1-\beta_m(q))
                  + (1-p_{mk})\beta_m(q))}{\beta_m(q)(1-\beta_m(q'))
                  + (1-\beta_m(q))\beta_m(q')}
      \\ \notag & \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad
                  + 1_{G_m=0} \frac{(1-p_{mk})}{1-\beta_m(q')}\Big)q_k'.
    \end{align}
    %As carried out in the SI, this function can be maximized by computing $\hat q^M, \hat q^P$ such that $\hat q^P = f(\hat q^M, \hat q^P)$ and $\hat q^M = f(\hat q^P, \hat q^M)$ for $f(q,q') = (f_k(q, q'))_{k=1,...,K}$ with (see \eqref{eqSI:fixedRecent} in the SI) \begin{align}\label{eq:fixedRecent}
%  f_k(q, q') & := \frac{1}{M}\sum_{m=1}^M \delta_k(q,q', G_m) q_k',
%  \\
%  \notag
%  \delta_k(q, q', g) & = \begin{cases}
%    \displaystyle\frac{p_{mk}}{\beta_m(q')}, & \text{ if }g=2,\\[2ex]
%    \displaystyle\frac{(p_{mk} (1-\beta_m(q))
%      + (1-p_{mk})\beta_m(q))}{\beta_m(q)(1-\beta_m(q'))
%      + (1-\beta_m(q))\beta_m(q')}, & \text{ if }g=1,\\[2ex]
%    \displaystyle\frac{(1-p_{mk})}{1-\beta_m(q')}, & \text{ if }g=0. \end{cases}
%\end{align}
%In our implementation, we iteratively compute $q_{n+1}^P = f(q_n^M, q_n^P)$ and $q_{n+1}^M = f(q_{n}^P, q_n^M)$ until convergence.

  \end{enumerate}
\end{remark}

\begin{proof}[Proof of Lemma~\ref{l2}]
  Again, we use Lagrange multipliers. Since
  \begin{align*}
    \frac{\partial \ell(q^M, q^P|G)}{\partial q^P_k}
    & = \sum_{m=1}^M \sum_l \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}},
    \\
    \frac{\partial \ell(q^M, q^P|G)}{\partial q^M_k}
    & = \sum_{m=1}^M \sum_l \frac{\alpha_{mkl}q_l^P}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}},
  \end{align*}
  we have to solve the system of equations
  \begin{align*}
    \lambda & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}}, \qquad k=1,...,K,\\
    \rho & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^P}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}}, \qquad k=1,...,K,
    \\
    1 & = \sum_{k=1}^K q^P_k = \sum_{k=1}^K q^M_k.
  \end{align*}
  It is easy to eliminate $\lambda$ and $\rho$, since
  \begin{align*}
    \lambda & = \lambda \sum_{k=1}^K q^P_k = \sum_{m=1}^M \sum_{k,l}
              \frac{\alpha_{mkl}q^P_kq_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q_{l'}^M} = M,
    \\
    \rho & = \rho \sum_{k=1}^K q_k^M = \sum_{m=1}^M \sum_{k,l}
           \frac{\alpha_{mkl}q^P_kq_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q_{l'}^M} = M.    
  \end{align*}
  So, we are left with finding $q^P$ and $q^M$ such that
  \begin{align*}
    \frac{1}{M}\sum_{m=1}^M \frac{\alpha_{mkl}q_l^P}{\sum_{k',l'}^K \alpha_{mk'l'}q^P_{k'} q^M_{l'}} & = 1, \qquad k=1,...,K,
    \\ \frac{1}{M}\sum_{m=1}^M \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'}^K \alpha_{mk'l'}q^P_{k'} q^M_{l'}} & = 1, \qquad k=1,...,K.
  \end{align*}
\end{proof}

\newpage

\section{Additional results}

\subsection{Some showcases from simulations}
\enlargethispage{0cm}
\begin{figure}[H]
  \begin{center}
    \parbox[b]{0.9\textwidth}{\includegraphics[width=0.95\textwidth]{abc_StructurePlot.pdf}\vspace{-2cm}}
  \end{center}
  \caption{\label{Sfig:sim2} Structure barplots for all 400 recently
    $A\times C$-admixed individuals. For the recent-admixture model, we display $\tfrac 12(\widehat q^M + \widehat q^P)$. The arrow indicates the
    individual from Figure~\ref{fig:sim1}.}
\end{figure}

\begin{figure}[H]
  \begin{center}
    \parbox[b]{0.5\textwidth}{\includegraphics[width=0.65\textwidth]{deviations_abc.pdf}\vspace{0cm}}
  \end{center}
  \caption{\label{Sfig:sim3} For all 3$\times$400 non-admixed samples,
    and 3$\times$400 admixed samples, we compute the errors by the
    distance to the true IA as given in \eqref{eq:error}.}
\end{figure}


\newpage

\subsection{Estimation accuracy}
For non-admixed and first generation admixed individuals, all cases (parents come from different continents) are described in the main
text. For second generation admixed individuals, we have several
cases, depending on the origin of the grand-parents.  When data from
four populations (AFR, EAS, EUR, SAS; see below) is available, we have
the following cases:
\begin{itemize}
\item[(A)] 4 non-admixed cases with IA 100:0: \\
  AFR/AFR$\times$AFR/AFR, EAS/EAS$\times$EAS/EAS,
  EUR/EUR$\times$EUR/EUR, SAS/SAS$\times$SAS/SAS;
\item[(B)] 6 admixed cases with IA 50:50, where both
  parents are non-admixed:\\
  AFR/AFR$\times$EAS/EAS, AFR/AFR$\times$EUR/EUR,
  AFR/AFR$\times$SAS/SAS, EAS/EAS$\times$EUR/EUR,
  EAS/EAS$\times$SAS/SAS, EUR/EUR$\times$SAS/SAS;
\item[(C)] 6 admixed cases with IA 50:50, where both
  parents are admixed:\\
  AFR/EAS$\times$AFR/EAS, AFR/EUR$\times$AFR/EUR,
  AFR/SAS$\times$AFR/SAS, EAS/EUR$\times$EAS/EUR,
  EAS/SAS$\times$EAS/SAS, EUR/SAS$\times$EUR/SAS;
\item[(D)] 12 admixed cases with IA 75:25:\\
  AFR/AFR$\!\times$AFR/EAS, AFR/AFR$\times$AFR/EUR,
  AFR/AFR$\!\times$AFR/SAS, EAS/EAS$\times$EAS/AFR,
  EAS/EAS$\!\times$EAS/EUR, EAS/EAS$\times$EAS/SAS,
  EUR/EUR$\!\times$EUR/AFR, EUR/EUR$\times$EUR/EAS,
  EUR/EUR$\!\times$EUR/SAS, SAS/SAS$\times$SAS/AFR,
  SAS/SAS$\!\times$SAS/EAS, SAS/SAS$\times$SAS/EUR;
\item[(E)] 12 second generation admixed with IA 50:25:25,
  where one parent is non-admixed:\\
  AFR/AFR$\times$EAS/EUR, AFR/AFR$\times$EAS/SAS,
  AFR/AFR$\times$EUR/SAS, EAS/EAS$\times$AFR/EUR,
  EAS/EAS$\times$AFR/SAS, EAS/EAS$\times$EUR/SAS,
  EUR/EUR$\times$AFR/EAS, EUR/EUR$\times$AFR/SAS,
  EUR/EUR$\times$EAS/SAS, SAS/SAS$\times$AFR/EAS,
  SAS/SAS$\times$AFR/EUR, SAS/SAS$\times$EAS/EUR;
\item[(F)] 12 second generation admixed with IA 50:25:25,
  where both parents are admixed:\\
  AFR/EAS$\times$AFR/EUR, AFR/EAS$\times$AFR/SAS,
  AFR/EUR$\times$AFR/SAS, EAS/AFR$\times$EAS/EUR,
  EAS/AFR$\times$EAS/SAS, EAS/EUR$\times$EAS/SAS,
  EUR/AFR$\times$EUR/EAS, EUR/AFR$\times$EUR/SAS,
  EUR/EAS$\times$EUR/SAS, SAS/AFR$\times$SAS/EAS,
  SAS/AFR$\times$SAS/EUR, SAS/EAS$\times$SAS/EUR;
\item[(G)] 3 second generation admixed with IA
  25:25:25:25:\\
  AFR/EAS$\times$EUR/SAS, AFR/EUR$\times$EAS/SAS,
  AFR/SAS$\times$EAS/EUR;
\end{itemize}
As can be seen in Figure~\ref{fig:mixed_allcases_EUROFORGEN}, the
recent-admixture model on average does not give worse estimates than the admixture
model, and outperforms the admixture-model in several cases; see also
Figure~\ref{fig:mixed_cases_EUROFORGEN} in the main text. For the
Kidd AIMset, see Figures~\ref{fig:mixed_cases_Kidd}
and~\ref{fig:mixed_allcases_Kidd}.

\subsubsection*{EUROFORGEN AIMset}

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{deviations_nonmixed_EUROFORGEN.pdf}
  \end{center}
  \caption{\label{fig:nonmixed_EUROFORGEN} For all non-admixed samples
    from the 1000 genomes dataset, we computed IA from the admixture
    and recent-admixture model. The distance to the true IA is
    computed as in \eqref{eq:error}.}
\end{figure}

\begin{center}
  \begin{turn}{270}%[htbp]
    \begin{minipage}{\textheight}
      \includegraphics[width=\textwidth]{deviations_mixed_allcases_EUROFORGEN.pdf}
      \captionof{figure}{For all cases of second generation admixed individuals, we
        compare estimation accuracy of IA using the EUROFORGEN AIMset.}
      \label{fig:mixed_allcases_EUROFORGEN}
    \end{minipage}
  \end{turn}  
\end{center}
       
\subsubsection*{Kidd AIMset}
For the Kidd AIMset, recent-admixture produces an error in the
non-admixed samples (see also Figure~\ref{fig:nonmixed_Kidd}), which
is smaller than the error for the admixture-model in 1394 out of 2157
cases, so the hypothesis that the error is worse for recent-admixture
can be rejected ($p<0.001$). % $ p < 10^{-34}$).
For first order admixed samples, 2381 out of 3000 cases have a smaller
error under recent-admixture (which gives $p<0.001$; %$p<10^{-241}$);
see also Figure~\ref{fig:mixed_cases_Kidd}.A. Last, for
second-generation admixed individuals, recent-admixture is more
accurate in 16116 out of 27500 cases ($p<0.001$); % $p<10^{-180}$);
see also Figures~\ref{fig:mixed_cases_Kidd}.B and
\ref{fig:mixed_allcases_Kidd}.

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{deviations_nonmixed_Kidd.pdf}
  \end{center}
  \caption{\label{fig:nonmixed_Kidd} Same as in
    Figure~\ref{fig:nonmixed_EUROFORGEN}, but for the Kidd AIMset.}
\end{figure}



\begin{figure}[H]
  \hspace{3cm} (A) \hspace{8cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_Kidd.pdf}\vspace{2cm}}
    \hspace{1cm}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_cases_Kidd.pdf}}
  \end{center}
  \caption{\label{fig:mixed_cases_Kidd} Same as in
    Figure~\ref{fig:mixed_cases_EUROFORGEN}, but using the Kidd
    AIMset.}
\end{figure}

%\begin{table}[H]
%  \centering
%  \input{power_table_Kidd.tex}
%  \caption{Same as Table~\ref{tab:power}, but using the Kidd AIMset.}
%  \label{tab:power_Kidd}
%\end{table}

\begin{center}
  \begin{turn}{270}%[htbp]
    \begin{minipage}{\textheight}
      \includegraphics[width=\textwidth]{deviations_mixed_allcases_Kidd.pdf}
      \captionof{figure}{Same as in
        Figure~\ref{fig:mixed_allcases_EUROFORGEN}, but using the Kidd
        AIMset.}
      \label{fig:mixed_allcases_Kidd}
    \end{minipage}
  \end{turn}
\end{center}

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{roc-curve-Kidd.pdf}
  \end{center}
  \caption{Same as in Figure~\ref{fig:ROC_EUROFORGEN}, but using the
    Kidd AIMset.}
  \label{fig:ROC_Kidd} 
\end{figure}

\subsection{Simulating $p$-values and hidden structure}

\begin{figure}[htb]
  \hspace{3cm} (A) \hspace{7cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{qvalues_EUROFORGEN_1ksim_H0.pdf}}    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{qvalues_Kidd_1ksim_H0.pdf}}
  \end{center}
  \caption{\label{fig:qvalues} Quantile-quantile plot of $\Delta\ell$ against a $\chi^2$ distribution with three degrees of freedom. AFR,... are simulated under $q_{\text{AFR}}=1$,... from the allele frequencies. AFR+EAS,... are simulated under $q_{\text{AFR}}=q_{\text{EAS}}=0.5$,... For Dirichlet-distributed $q$, we have the mixed case. The distribution of $\Delta\ell$ not only is far from $\chi^2(3)$-distributed, but also depends on $q$. (A) For the EUROFORGEN AIMset with classes AFR, EAS, EUR and SAS; (B) For the Kidd AIMset.}
\end{figure}

\begin{figure}[htb]
  \hspace{3cm} (A) \hspace{7cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{pvalues_EUROFORGEN_1ksim_H0.pdf}}    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{pvalues_Kidd_1ksim_H0.pdf}}
  \end{center}
  \caption{\label{fig:pvalues} Distribution of $p$-values for several types of individuals. Data used is as in Figure \ref{fig:qvalues}. If the curve is below $f(x) = x$, we are facing a conservative test, since there is at most a fraction $x$ of $p$-values below $x$. (A) For the EUROFORGEN AIMset with classes AFR, EAS, EUR and SAS; (B) For the Kidd AIMset.}
\end{figure}

\begin{figure}[htb]
  \hspace{3cm} (A) \hspace{7cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{pvalues_EUROFORGEN_hiddenstructure.pdf}}    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{pvalues_Kidd_hiddenstructure.pdf}}
  \end{center}
  \caption{\label{fig:hiddenstructure} Same as in Figure~\ref{fig:pvalues}, but EAS and SAS was combined in the reference database, i.e.\ it has hidden structure. (A) For the EUROFORGEN AIMset. (B) For the Kidd AIMset.}
\end{figure}

\newpage\mbox{}
\newpage


\subsection{AIMs analysis of two collected individuals with recent admixture}

\begin{figure}[H]
  \parbox[b]{\textwidth}{\includegraphics[width=0.9\textwidth]{illumina_freqs.pdf}}
  \caption{\label{Sfig:allFreqs} Allele frequencies for five populations for all 53 AIMs from the Forensic {\em MPS AIMs Panel Reference Sets}, taken from \url{http://mathgene.usc.es/snipper/illumina_55.xlsx}, which are used in the analysis of the collected individuals.}
\end{figure}

\includepdf[pages=-]{all_outlier_1k}
\includepdf[pages=-]{all_outlier_self}


\bibliographystyle{SI}{unsrt}
\bibliography{SI}{BGA}{References}

\newpage

\thispagestyle{empty}
\section*{Highlights}
\begin{itemize}
    %\item The excess of heterozygote sites in recently admixed individuals (known as the Wahlund principle) can introduce bias in the analysis of Biogeographical Ancestry (BGA). 
    \item We improve statistical methods as used in {\sc STRUCTURE} and {\sc ADMIXTURE} for Biogeographical Ancestry (BGA) inference to account for recent admixture, i.e. different admixture of both parents.
    \item The resulting recent admixture model has a higher accuracy in estimating individual admixture in most cases. 
    \item We give a likelihood ratio test for recent admixture, which is both highly specific and sensitive for recent admixture.
    \item We find evidence of recent admixture in the 1000 genomes dataset.
    \item The self-report on recent admixture of two self-sequenced samples was {\color{blue} only confirmed in one case.}
    \end{itemize}
\end{document}













