\documentclass[12pt]{article}
\usepackage[utf8]{inputenc}
\usepackage{multibbl}
\usepackage{hyperref}
\newbibliography{article}
\newbibliography{SI}

% Usage of multibbl:
% bibtex artile
% bibtex SI

%\usepackage[sectionbib,numbers]{natbib}
\usepackage{eurosym}
\usepackage{float}
\usepackage{amstext,amsthm}
\usepackage{amsmath}
\usepackage{amssymb,mathtools}
\usepackage[nointegrals]{wasysym} % enthaelt iint-Def die aber auch in amsmath definiert sind
\usepackage{mathrsfs}
%\usepackage{refcheck}
\usepackage{graphicx}
\usepackage[figuresleft]{rotating}
\usepackage{rotating, float, caption}


\usepackage{tikz}
%\usepackage{enumerate}

\newtheorem{proposition}{Proposition}[section]
\newtheorem{corollary}[proposition]{Corollary}
\newtheorem{theorem}{Theorem}
\newtheorem{lemma}{Lemma}[section]
\theoremstyle{definition}
\newtheorem{definition}{Definition}[section]
\newtheorem{remark}{Remark}[section]
\newtheorem{example}{Example}[section]
\newtheorem{estimator}{Estimator}[section]
\pagestyle{headings}

% \setlength{\bibsep}{0pt}

\setlength{\textheight}{230mm}
\setlength{\topmargin}{-15mm} 
\setlength{\textwidth}{17cm} 
\setlength{\oddsidemargin}{-3mm}
\setlength{\columnseprule}{.1pt}
\setlength{\columnsep}{20pt}

\def\blstr{1.1}
\renewcommand{\baselinestretch}{\blstr}
\def\blstrtable{1.55}
\def\blstrcode{1.2}  
%\renewcommand{\labelenumi}{\alph{enumi})}
%%\setlength{\oddsidemargin}{0.28in} 
%%\setlength{\evensidemargin}{0.28in}

%\addtolength{\bibsep}{1.5mm}
%%\bibpunct{(}{)}{,}{a}{}{,}

%%\RequirePackage{ae,mathpple}
%%\renewcommand{\rmdefault}{ppl}
%%\renewcommand{\sfdefault}{aess}
%%\renewcommand{\ttdefault}{aett}
\newcommand{\Rpackage}[1]{{\normalfont\fontseries{b}\selectfont #1}}

%\usepackage{a4}
\usepackage[normalem]{ulem}
\usepackage{color}
\newcommand\crule[3][black]{\textcolor{#1}{\rule{#2}{#3}}}

\newcommand{\dirk}[3]{\textcolor{red}{\sout{#1}\textcolor{blue}{#2}}{\begin{marginpar}{\raggedright\color{red} Dirk:\tiny {#3}}\end{marginpar}}}
\newcommand{\peter}[3]{\textcolor{red}{\sout{#1}\textcolor{blue}{#2}}{\begin{marginpar}{\raggedright\color{green} Peter:\tiny {#3}}\end{marginpar}}}


\DeclareMathAlphabet{\mathpzc}{OT1}{pzc}{m}{it}
\newcommand{\smallu}{\mathpzc{u}}
\newcommand{\smallx}{\mathpzc{x}}
\newcommand{\smally}{\mathpzc{y}}
\newcommand{\smallz}{\mathpzc{z}}
\newcommand{\ind}[1]{\mathbbm{1}_{\{#1\}}} %Definition Indikatorfunktion
\newcommand{\indset}[1]{\mathbbm{1}_{#1}} %Definition Indikatorfunktion
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}


\usepackage{times}
%%\RequirePackage[T1]{fontenc}

\begin{document}
\title{\LARGE Inference of recent admixture using genotype data}

\author{\sc Peter Pfaffelhuber, Elisabeth Huss, Franz Baumdicker, \\
  \sc Sabine Lutz-Bonengel, Fabian Staubach}

\date{\today}

\maketitle

{\color{blue}
{\sc todo}
\begin{enumerate}
\item What are we doing with the Tvedebrink-tests?
  % assigns first
  % generation admixed individuals wrongly to admixed populations (FS)
\item Local ancestry with tsinfer;
\item Analyse data from Denise;
\end{enumerate}
}

\begin{abstract}
  \noindent
  For inference of individual genetic histories, admixture barplots
  are being used abundantly in forensic genetics. These plots
  visualize parameters for individual ancestry (IA), as inferred from
  the admixture-model, as e.g.\ implemented in the software {\sc
    structure} and {\sc admixture}. In this model, it is assumed that
  every allele in the individual's genome originates in one of several
  ancestral populations with the same probability. We will showcase
  that estimates of IA might be inaccurate in cases of recent
  adxmixture. As a way out, we introduce the recent-admixture model,
  which makes use of the excess of heterozygotes observed in recently
  admixed individuals. In this model, we assume that the two
  homologous copies of each allele originate from the ancestral
  populations independently for the mother's and the father's
  copy. Estimates for IA in absence of recent admixture are almost
  identical for the admixture and recent admixture model. However,
  they are more accurate for the recent-admixture model in individuals
  which are in fact recently admixed. Moreover, we develop a
  likelihood ratio test for recent admixture, which has a high power
  to find recently admixed individuals. We analyse data from the 1000
  genomes dataset with our methods and find some recently admixed
  individuals.
\end{abstract}

\section{Introduction}
Inference of the geographical ancestry of a trace using genetic
markers is today a well-established research field in forensic
genetics (see e.g.\ \cite{article}{Phillips2016, Eduardoff2016,
  Kidd2017}). Either the trace is classified into one of several
groups of different origin (e.g.\ Africa, Europe, East Asia, Native
America, and Oceania; see e.g.\ \cite{article}{Snipper2007,
  Pfaffelhuber2019}, or it is assumed that it consists of a mixture of
ancestral genetic material originating in several groups. For this
task, {\sc structure} \cite{article}{Pritchard2000} has become the
de-facto standard to estimate individual ancestry (IA) proportions of
a trace among several continental groups. This Bayesian approach using
MCMC was complemented by the faster, likelihood based approach,
implemented in the software {\sc admixture}
\cite{article}{Alexander2009}; see also \cite{article}{Tang2005} for
the same model.

When using {\sc structure} or {\sc admixture} for estimating
continental (or other scales of geography) IA of a trace, allelic
frequencies for all continents from a reference database must be
used. A main assumption for estimating IAs is that all allelic states
have the same independent chance to share a copy from some continent.
When using a (small) set of ancestry informative markers, the large
genomic distance makes loci almost independent by recombination,
resulting in independence of markers across loci. However,
independence of the two copies at the same marker, usually denoted by
Hardy-Weinberg equilibrium, may not always hold. For example, consider
an individual whose parents have different continental backgrounds,
$A$ and $B$, say, and an allele which separates perfectly between the
two continents. Then, the individual will certainly be a heterozygote
at this marker. This increase in frequency for heterozygotes under
admixture is known for a long time and usually called the Wahlund
effect \cite{article}{Wahlund1928}.  However, {\sc structure} and {\sc
  admixture} will estimate that the chance for some allele to come
from $A$ and $B$ to be 50\%, which is then also be the estimated
probability for a heterozygote at the locus (due to the assumption of
independence of allelic states in the admixture model).

In general, {\sc structure} (and {\sc admixture}) are able to give
accurate estimates of IA also in the case of recent-admixture. As
observed in \cite{article}{Cheung2018}, {\sc structure} outperforms a
newly developed Genetic Distance Algorithm preforms in almost all
cases of recent admixture (except for the case when all four
grand-parents come from different populations). However, {\sc
  structure} does not reveal all relevant information in the data in
the case of recent admixture, since e.g.\ an IA of 50\% $A$ and 50\%
$B$ either indicates two parents with these IAs or a recent admixture
event of one individual from $A$ and one from $B$.

Our goal is to be able to distinguish these cases by making use of the
Wahlund effect, which will also result in better estimates of IA. So,
we aim for (i) improving estimates of IA and (ii) detecting recent
admixture. In particular, we extend the likelihood-model behind {\sc
  structure} or {\sc admixture} in order to account for recent
admixture. In the resulting recent-admixture-model, IA consists of two
vectors, one for each parent. We will call this pair of vectors
Parental Individual Admixture, PIA. So, in the above example, the
recent-admixture-model would estimate father and mother to have IAs as
100\% $A$ and 100\% $B$, respectively, due to the excess of
heterozygotes.  Since we are using only autosomal markers, we cannot
distinguish which of the parents comes from $A$ and $B$, though. Then,
with the estimate of PIA, the estimated probability for a heterozygote
is~1 in the example. Let us mention that in some occasions, {\sc
  STRUCTURE} draws wrong conclusions on DNA of recently admixed
individuals. We see in simulations, that a recently admixed from two
distant populations is estimated to come mainly from some intermediate
location.

The excess of heterozygotes was recently used to detect recent
admixture using statistical tests. For example,
\cite{article}{McNevin2019} uses a statistical test by simply counting
the number of heterozygous positions in an individual in order to
infer recent admixture. A different approach is used by
\cite{article}{Tvedebrink2018, Tvedebrink2019}. Here,
likelihood-ratio tests were developed which test the null-hypothesis
of non-admixture and recent (first generation) admixture versus the
alternative that the studied sample is not represented in the
reference database. We complement this test by observing that the
adxmixture model is a special case of the recent-admixture model, if
the IA of both parents is the same. This paves the way to give a
likelihood-ratio-test for recent admixture, with equality of the
parental IAs as null-hypothesis. The alternative hypothesis, however,
is that the studied sample shows recent admixture of populations
within the reference database.

As a showcase, we present data from a (female) individual whose DNA
was collected during a study in Freiburg, Germany. She self-reported
to have a mother from Germany and a father from Germany and a mother
from the Philippines. Here, when using a classification tool such as
{\sc Snipper}, her DNA is classified as South-East-Asian (with
reference samples from India, Pakistan etc). While the admixture model
correctly estimates ancestral proportions from Europe, East-Asia and
Oceania, the recent-admixture model clearly identifies a recently
admixed genome. In addition, the mt-DNA xxx.







\section{Materials and Methods}
We start by briefly recalling the admixture model, which is the basis
for the widely used sofware {\sc structure} \cite{article}{Pritchard2000}, {\sc
  admixture} \cite{article}{Alexander2009} and {\sc frappe} \cite{article}{Tang2005}.
Afterwards, we introduce a new model, called the recent-admixture
model. More details on the derivations in the admixture and
recent-admiture model can be found in the SI. Moreover, the
implementation of our methods can be downloaded from \url{
  https://github.com/pfaffelh/recent-admixture}. For both, the
adxmiture and recent-admixture model, we assume to have reference
database of $M$ bi-allelic markers from $K$ populations. However, from
this reference database, we only need to known allele frequencies,
i.e.\ by $p_{mk}$, the frequency of allele~1 at marker $m$ in
population $k$ for all $m=1,...,M$ and $k=1,...,K$. We have a trace
with $G_m \in \{0,1,2\}$ copies of allele~1 at marker $m$ for
$m=1,...,M$.  We will assume throughout that the allele frequencies
$p_{mk}$ are given and will not be changed by analysing the
trace. This is important since in currently used software {\sc
  structure}, {\sc admixuture} and {\sc frappe}, mostly in
non-forensic use, it is frequently the case that many new individuals
are studied, and allele frequencies are updated. For forensic use,
when analysing several traces at once, this would imply that the
results for the ancestry of trace~1 depend not only on the reference
data, but also on the data for traces~2, 3,... which seems
inappropriate. Hence, we do not make the computational overload of
updating allele frequencies, which would also lead to increased
runtimes and take the allele frequencies as given in the reference
database. With other words, we will follow the approach of
\cite{article}{Chakraborty1986} and \cite{article}{Hanis1986} for analysing our models.

\subsection{The admixture model}
Assuming that each allele observed in the trace comes from population
$k$ with probability $q_k$, the probability to observe allele~1 at
marker $m$ is
\begin{align}
  \label{eq:beta}
  \beta_m(q) := \sum_k p_{mk} q_k,  
\end{align}
and the log-likelihood of $q = (q_k)_{k=1,...,K}$ is (see
also~\eqref{Seq:lik} in the SI)
\begin{align}\label{eq:logLadmixture}
  \ell(q|G) = \sum_{m=1}^M \log\Big(\binom{2}{G_m} \beta_m(q)^{G_m}(1-\beta_m(q))^{2-G_m}\Big).
\end{align}
Assuming that all $p_{mk}$'s are known, this function can be maximized
over $q$ by computing $\hat q = (\hat q_k)_{k=1,...,K}$ such that
$\hat q_k = f_k(\hat q)$ for (see also \eqref{eqSI:fixed} in the SI)
\begin{align}\label{eq:fixed}
  f_k(q) =
  \frac{1}{2M} \sum_{m=1}^M \Big(G_m \frac{p_{mk}}{\beta_m(q)} + (2-G_m)\frac{1-p_{mk}}{1-\beta_m(q)}\Big)q_k,
  \qquad k =1,...,K.
\end{align}
This can be done numerically by iterating
$q_{n+1} = (f_k(q_n))_{k=1,...,K}$ until convergence. (In our
implementation, we continue the iteration until
$|q_{n+1} - q_n|< 10^{-6}$.) We note that this approach is essentially
the same as in the EM-algorithm from \cite{article}{Tang2005}, but
combining the expectation and maximization steps, since we do not
update allele frequencies. In addition, although maximizing
\eqref{eq:logLadmixture} could also be handled using a Newton method
as in \cite{article}{Alexander2009}, this approach has the advantage
that $q_n$'s are positive in all steps, and the sum of all entries in
$q_n$ is always~1. Moreover, the iteration is computationally fast if
only a small or moderate number of alleles is considered.

\subsection{The recent-admixture model}
When mother and father of an individual come with their own vectors of
admixture proportions, $q^M$ and $q^P$, the log-likelihood from
\eqref{eq:logLadmixture} changes to (see also
\eqref{eqSI:logLrecentadmixture} in the SI)
\begin{align}\label{eq:logLrecentadmixture}
  \ell(q^M, q^P|G) & = \sum_{m=1}^M \log( \gamma_m(q^M, q^P, G_m)),
  \\
  \notag
  \gamma_m(q^M, q^P, g) & = \begin{cases}\beta_m(q^M) \beta_m(q^P), & \text{ if }g=2,\\
    (\beta_m(q^M) (1-\beta_m(q^P))
    + (1-\beta_m(q^M)) \beta_m(q^P)), & \text{ if }g=1,\\
    (1-\beta_m(q^M))(1-\beta_m(q^P)), & \text{ if }g=0. \end{cases}
\end{align}
\sloppy As carried out in the SI, this function can be maximized by
computing $\hat q^M, \hat q^P$ such that
$\hat q^P = f(\hat q^M, \hat q^P)$ and
$\hat q^M = f(\hat q^P, \hat q^M)$ for
$f(q,q') = (f_k(q, q'))_{k=1,...,K}$ with (see
\eqref{eqSI:fixedRecent} in the SI)
\begin{align}\label{eq:fixedRecent}
  f_k(q, q') & := \frac{1}{M}\sum_{m=1}^M \delta_k(q,q', G_m) q_k',
  \\
  \notag
  \delta_k(q^M, q^P, g) & = \begin{cases}
    \displaystyle\frac{p_{mk}}{\beta_m(q')}, & \text{ if }g=2,\\[2ex]
    \displaystyle\frac{(p_{mk} (1-\beta_m(q))
      + (1-p_{mk})\beta_m(q))}{\beta_m(q)(1-\beta_m(q'))
      + (1-\beta_m(q))\beta_m(q')}, & \text{ if }g=1,\\[2ex]
    \displaystyle\frac{(1-p_{mk})}{1-\beta_m(q')}, & \text{ if }g=0. \end{cases}
\end{align}
In our implementation, we iteratively compute
$q_{n+1}^P = f(q_n^M, q_n^P)$ and $q_{n+1}^M = f(q_{n+1}^P, q_n^M)$
until convergence.

\subsection{Obtaining admixed individuals in silico}
\label{S:insilico}
In order to test our method, we created admixed individuals from a
reference database. (We use the 1000 genomes dataset, but excluding
Admixed Americans, AMR; see below.) For example, we obtain an
individual admixed from populations $k$ and $k'$ by choosing a genome
$\tilde G = (\tilde G_m)_{m=1,...,M}$ from population $k$ and
$\bar G = (\bar G_m)_{m=1,...,M}$ from population $k'$ as the
parents. Then, $G_m = X_m + Y_m$ where $X_m = 1$ with probability
$\tilde G_m/2$, $X_m=0$ with probability $(2-\tilde G_m)/2$ and
$Y_m = 1$ with probability $\bar G_m/2$, $Y_m=0$ with probability
$(2-\bar G_m)/2$. When iterating this procedure, we can also model
second-order admixed individuals etc.\ in silico.

Using AFR, EAS, EUR, SAS as population labels (as in the 1000 genomes
dataset), all cases for second generation admixed individuals fall
into one of seven categories. Writing up the ancestries of the four
grand-parents {\it Mother of mother/father of mother$\times$mother of
  father/father of father}, we have the following distinguishable
cases for second generation admixed individuals (the full list of all
resulting 55 cases is given in the SI; note that \cite{article}{Cheung2018}
come up with only 35 cases, since they do not distinguish between
maternal and paternal ancestry, e.g.\ they count
AFR/AFR$\times$EAS/EAS and AFR/EAS$\times$AFR/EAS as one case):

\begin{itemize}
\item[(A)] 4 non-admixed cases, e.g.\ AFR/AFR$\times$AFR/AFR;
\item[(B)] 6 admixed cases with admixture ratio 50:50, where both
  parents are non-admixed, e.g. AFR/AFR$\times$EAS/EAS;
\item[(C)] 6 admixed cases with admixture ratio 50:50, where both
  parents are admixed, e.g.\ AFR/EAS$\times$AFR/EAS;
\item[(D)] 12 admixed cases with admixture ratio 75:25, e.g.\
  AFR/AFR$\times$AFR/EAS;
\item[(E)] 12 admixed cases with admixture ratio 50:25:25, where one
  parent is non-admixed, e.g.\ AFR/AFR$\times$EAS/EUR;
\item[(F)] 12 admixed with admixture ratio 50:25:25, where both
  parents are admixed, e.g.\ AFR/EAS$\times$AFR/EUR;
\item[(G)] 3 admixed with admixture ratio 25:25:25:25, e.g.\
  AFR/EAS$\times$EUR/SAS;
\end{itemize}
For each of the other 55 cases, we simulated 500 individuals in silico
by picking four grand-parents at random from the populations, creating
mother and father from the grand-parents, and creating a new
individual from the parents, as described above.

\subsection{Comparing results from admixture and recent-admixture}
For a reference database from which we compute (or estimate) allele
frequencies $p_{mk}$ (which is the allele frequency of allele~1 at
marker $m$ in population $k$), we can estimate $q$ from the admixture
model as well as $q^M,q^P$ from the recent-admixture model, as
described in \eqref{eq:fixed} and \eqref{eq:fixedRecent}. In order to
compare the results from the admixture and recent-admixture model, we
compute $q_k^{MP} := \tfrac 12 (q_k^M + q_k^P)$ for $k=1,...,K$, which
give the fractions of the genome coming from populations
$1,...,K$. Then, for a non-admixed individual, we have
$q_k^{\text{\sc true}} = 1$ for some $k$, and for an admixed
individual with parents from populations $k$ and $k'$ we have
$q_k^{\text{\sc true}} =q_{k'}^{\text{\sc true}} = 0.5$, and similarly
for individuals with grandparents from two up to four different
populations. Computing the estimation error, i.e.\ the {\em distance
  to the true IA} for the admixture model, results in
\begin{align}
  \label{eq:error}
  \sum_k |q_k - q_k^{\text{\sc true}}| \text{  and }\sum_k |q_k^{MP} - q_k^{\text{\sc true}}|
\end{align}
for the recent-admixture model. We stress that in the recent-admixture
model, we in fact obtain results for $q^M$ and $q^P$ separately, such
that even more information than $q^{MP}$ is contained in the estimates
for this model.

\subsection{A likelihood-ratio test for recent admixture}
We want to test if data $G = (G_m)_{m=1,...,M}$ from a new trace fits
significantly better to the recent-admixture model than to the
admixture model. Since the admixutre model is identical to the
recent-admixture model for $q^M = q^P = q$, we are testing the
hypothesis $H_0: q^M = q^P$ against $H_1: q^M\neq q^P$. For this, we
take the estimators $\hat q$ of $q$ from iteration of
\eqref{eq:fixed}, and $\hat q^M, \hat q^P$ of $q^M$ and $q^P$ from
iteration of \eqref{eq:fixedRecent} and compute
\begin{align}
  \label{eq:DeltaEll}
  \Delta\ell := \ell(\hat q^M, \hat q^P|G) - \ell(\hat q|G)
\end{align}
with
$\ell(q^M, q^P|G)$ from \eqref{eq:logLrecentadmixture} and $\ell(q|G)$
from \eqref{eq:logLadmixture}. If $\Delta \ell >x$ for some $x$ (which
needs to be specified), the recent-admixture model fits significantly
better and we reject $H_0$. If $\Delta \ell \leq x$, we accept
$H_0$. In order to find $x$, we fix a $p$-value (0.01, say), and take
$x$ to be the $p$-quantile of the empirical distribution for data in
the reference database. (This means, if $p=0.01$ and the reference
dataset contains 1000 samples, we compute all values for $\Delta\ell$
for all samples, and set $x$ to be the 10th-smallest value we
obtained.)

\subsection{Reference data}
\sloppy We downloaded 1000 Genomes data (phase 3) from
\url{ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/}, as
well as information on the sampling locations from
\url{ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/integrated_call_samples_v3.20130502.ALL.panel}
\cite{article}{Auton2015}. This is data from 661 individuals from Africa (AFR),
347 Admixed Americans (AMR), 504 East Asians (EAS), 503 Europeans
(EUR) and 489 South Asians (SAS). The dataset comes with approximately
80 million SNPs. However, we use only a few of them known as the
EUROFORGEN AIMset \cite{article}{Phillips2014} and Kidd AIMset \cite{article}{Kidd2014},
respectively. The former comes with 128 SNPs, and we ignore seven
tri-allelic SNPs (rs17287498, rs2069945, rs2184030, rs433342,
rs4540055, rs5030240, rs12402499), since our methods currently rely on
bi-allelic SNPs. It was designed to distinguish Africa, Europe, East
Asia, Native America, and Oceania, but was shown to perform well on
the 1000 genomes dataset, also for distinguishing South Asia, even
when ignoring the tri-allelic SNPs \cite{article}{Pfaffelhuber2019}.  The
latter comes with 55 bi-allelic SNPs and was introduced as a global
AIMset differentiating between 73 populations. We note that this
AIMset is part of the Verogen MiSeq FGxâ„¢ Forensic Genomics Solution.

\subsection{A sample from Freiburg xxx}

\section{Results}

\subsection{Some showcases from simulations}
We simulated genome-wide data from a sample, taken from a population
genetic model with three islands, $A$, $B$ and $C$, using
\cite{article}{Kelleher2016}. Migration is such that only $A, B$ and $B, C$ are
connected, but not $A, C$. We used a migration rate of 10 diploid
individuals per connected islands (in both directions) per
generation. More precisely, we simulate a sample of 400 individuals
per island, each with 20~recombining chromosomes, each with about
2.5$\cdot 10^4$ SNPs. From these $\sim 5\cdot 10^5$ SNPs, we use the
step-wise approach from \cite{article}{Pfaffelhuber2019} to look for~10
Ancestry Informative Markers (AIMs). When using a naive Bayes approach
as in {\sc snipper} \cite{article}{Snipper2007}, this AIMset gives a vanishing
misclassification error for the task of classifying the 3$\times$400
simulated, non-admixed individuals.

Subsequently, we used the admixture and recent-admixture model to
estimate IA (and PIA) for both, non-admixed and recently (first
generation) admixed individuals. The latter were obtained as described
in Section~\ref{S:insilico}. We observe that the admixture model fails
to give accurate estimates for IA in $A\times C$-recently-admixed
individuals for two reasons. First, it correctly predicts that the
individual is $A\times C$-admixed, but overestimates one of the two
ancestral proportions. Second, and more severely, it confounds the
signal for recent admixture with an ancestral proportion from island
$B$. In both cases, the recent-admixture model frequently gives better
estimates for IA. Figure~\ref{fig:sim1} shows examples for these two
cases, which are taken from all $A\times C$-admixed individuals as
displayed in Figure~\ref{Sfig:sim2}.

To get an overall picture, we computed errors for estimating IA as
given in \eqref{eq:error} for the admixture and recent-admixture
model. As described in MM, we average estimates $\hat q^M$ and
$\hat q^P$ from the recent-admixture model, in order to compare to the
true IAs. Figure~\ref{Sfig:sim3} displays these errors in all cases
including non-admixed individuals, and all three cases of
recent-admixture. Interestingly, binomial tests with the alternative
that the recent-admixture model gives smaller errors show significant
results in all but one case ($A, B, A\times B, A\times C, B\times C$:
$p<0.001$, $C: p=0.14$). This shows that on average the
recent-admixture model performs better than the admixture model, even
on non-admixed samples.

\begin{figure}
  \parbox[t]{0.2\textwidth}{  
    \hspace{1cm} (A)

  ~
  
  \begin{tikzpicture}[every node/.style={circle}]
    \node[fill = blue, inner sep=.01cm] (n1) at (1,2.5)   {\color{white}$\;A\;\,$};
    \node[fill = green, inner sep=.01cm] (n2) at (2,2.5) {$\;B\;\,$};
    \node[fill = yellow, inner sep=.01cm] (n3) at (3,2.5) {$\;C\;\,$};
    \foreach \from/\to in
    {n1/n2,n2/n3}
    \draw[<->] (\from) -- (\to);
  \end{tikzpicture}
  }   \parbox[t]{0.42\textwidth}{  
    \hspace{2.8cm} (B)

  ~

  % seed 210, mixed A x C 202
  \begin{tabular}{|l|r|}\hline
    \rule[-3ex]{0cm}{7ex} \parbox{1.8cm}{true}
    \hspace{0cm} \parbox{1cm}{\hfill $q^{MP}$ \ $q^M, q^P$}
    & \parbox{2cm}{\begin{tikzpicture} 
        \draw[fill=blue] (0cm, 0cm) rectangle (1cm, .4cm);
        \draw[fill=green] (1cm, 0cm) rectangle (1cm, .4cm);
        \draw[fill=yellow] (1cm, 0cm) rectangle (2cm, .4cm);
        \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
      \end{tikzpicture}
    \\[-1.3ex]\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=green] (2cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    \\[-2ex]\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
      \draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
      \draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    }
    \\ \hline\hline
    \rule[-1ex]{0cm}{4ex} admixture \hspace{1cm} \parbox{.5cm}{$\hat q$ \\[-1.3ex] \mbox{}}
    &  \begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (0.334cm, .4cm);
      \draw[fill=green] (0.334cm, 0cm) rectangle (1.632cm, .4cm);
      \draw[fill=yellow] (1.632cm, 0cm) rectangle (2cm, .4cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
    \end{tikzpicture}
    \\ \hline
    \rule[-3ex]{0cm}{7ex} \parbox{1.8cm}{recent-\\admixture}
    \hspace{0cm} \parbox{1cm}{\hfill $\hat q^{MP}$ \\[1ex]  $\hat q^M, \hat q^P$}
    & \parbox{2cm}{\begin{tikzpicture} 
        \draw[fill=blue] (0cm, 0cm) rectangle (0.848cm, .4cm);
        \draw[fill=green] (0.848cm, 0cm) rectangle (1.208cm, .4cm);
        \draw[fill=yellow] (1.208cm, 0cm) rectangle (2cm, .4cm);
        \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
      \end{tikzpicture}
    \\[-1.3ex]\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (1.696cm, .2cm);
      \draw[fill=green] (1.696cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    \\[-2ex]\begin{tikzpicture} 
      \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
      \draw[fill=green] (0cm, 0cm) rectangle (0.414cm, .2cm);
      \draw[fill=yellow] (0.414cm, 0cm) rectangle (2cm, .2cm);
      \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
    \end{tikzpicture}
    }\\ \hline
  \end{tabular}
  }   \parbox[t]{0.32\textwidth}{  
    \hspace{2cm}(C)

  \includegraphics[width=0.3\textwidth]{abc_baryplot2.pdf}}
  
\caption{(A) Illustration of the population model for the
  simulations. Islands $A, B$ and $B, C$ are connected trough 10
  migrants per generation, but $A, C$ is not directly connected. (B)
  When estimating IA and PIA, an $A\times C$ admixed individual is
  found to have most of its ancestry in $B$ when using admixture,
  since allele frequencies in $B$ are between $A$ and $C$.  However,
  recent-admixture correctly predicts two parents of different
  ancestry, one mostly $A$, the other mostly $C$. (C) When trying to
  classify individuals into $A$, $B$ or $C$, blue dots correspond to
  (non-admixed) individuals from $A$, yellow dots to $C$ and black
  dots to $A\times C$ individuals. Altough all non-admixed individuals
  are correctly assigned, admixed individuals are mostly not even
  assigned to $A$ or $C$. The individual from (B) is indicated by the
  arrow and is assigned to $B$ with probability almost 100\%.}
  \label{fig:sim1}
\end{figure}

\begin{figure}[H]
  \begin{center}
    \parbox[b]{0.9\textwidth}{\includegraphics[width=0.95\textwidth]{abc_StructurePlot.pdf}\vspace{-2cm}}
  \end{center}
  \caption{\label{Sfig:sim2} Structure barplot for all 400 recently
    $A\times C$-admixed individuals. The arrow indicates the
    individual from Figure~\ref{fig:sim1}(B).}
\end{figure}

% \begin{figure}
%   \centering
%   % mixed A/C 177
%   \begin{tabular}{l|l|r|}\cline{2-3}
%     \rule[-3ex]{0cm}{7ex} & \parbox{1.8cm}{true}
%     \hspace{0cm} \parbox{1cm}{\hfill $q^{MP}$ \ $q^M, q^P$}
%     & \parbox{2cm}{\begin{tikzpicture} 
%         \draw[fill=blue] (0cm, 0cm) rectangle (1cm, .4cm);
%         \draw[fill=green] (1cm, 0cm) rectangle (1cm, .4cm);
%         \draw[fill=yellow] (1cm, 0cm) rectangle (2cm, .4cm);
%         \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
%       \end{tikzpicture}
%     \\[-1.3ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=green] (2cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     \\[-2ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     }
%     \\ \hline\hline
%     \parbox{.3cm}{
%     \begin{sideways}
%       \phantom{a}1
%     \end{sideways}}
%     \rule[-1ex]{0cm}{4ex} & admixture \hspace{1cm} \parbox{.5cm}{$\hat q$ \\[-1.3ex] \mbox{}}
%     &  \begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (0.428cm, .4cm);
%       \draw[fill=green] (0.428cm, 0cm) rectangle (0.428cm, .4cm);
%       \draw[fill=yellow] (0.428cm, 0cm) rectangle (2cm, .4cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
%     \end{tikzpicture}
%     \\ \cline{2-3}
%     \parbox{.3cm}{
%     \begin{sideways}
%       \phantom{a}Individual
%     \end{sideways}}
%     \rule[-3ex]{0cm}{7ex} & \parbox{1.8cm}{recent-\\admixture}
%     \hspace{0cm} \parbox{1cm}{\hfill $\hat q^{MP}$ \\[1ex]  $\hat q^M, \hat q^P$}
%     & \parbox{2cm}{\begin{tikzpicture} 
%         \draw[fill=blue] (0cm, 0cm) rectangle (0.998cm, .4cm);
%         \draw[fill=green] (0.998cm, 0cm) rectangle (1cm, .4cm);
%         \draw[fill=yellow] (1cm, 0cm) rectangle (2cm, .4cm);
%         \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
%       \end{tikzpicture}
%     \\[-1.3ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (1.998cm, .2cm);
%       \draw[fill=green] (1.998cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     \\[-2ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     }\\ \hline
%     \hline
%     \parbox{.3cm}{
%     \begin{sideways}
%       \phantom{a}2
%     \end{sideways}}
%     \rule[-1ex]{0cm}{4ex} & admixture \hspace{1cm} \parbox{.5cm}{$\hat q$ \\[-1.3ex] \mbox{}}
%     & \begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (0.342cm, .4cm);
%       \draw[fill=green] (0.342cm, 0cm) rectangle (1.114cm, .4cm);
%       \draw[fill=yellow] (1.114cm, 0cm) rectangle (2cm, .4cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
%     \end{tikzpicture}
%     \\ \cline{2-3}
%     \parbox{.3cm}{
%     \begin{sideways}
%       \phantom{a}Individual
%     \end{sideways}} 
%     \rule[-3ex]{0cm}{7ex}& \parbox{1.8cm}{recent-\\admixture} \hspace{0cm}
%   \parbox{1cm}{\hfill $\hat q^{MP}$ \\[1ex] $\hat q^M, \hat q^P$} &
%   \parbox{2cm}{\begin{tikzpicture} \draw[fill=blue] (0cm, 0cm)
%       rectangle (0.882cm, .4cm); \draw[fill=green] (0.882cm, 0cm)
%       rectangle (1cm, .4cm); \draw[fill=yellow] (1cm, 0cm) rectangle
%       (2cm, .4cm); \draw[fill=black] (2cm, 0cm) rectangle (2cm, .4cm);
%     \end{tikzpicture}
%     \\[-1.3ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (1.764cm, .2cm);
%       \draw[fill=green] (1.764cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=yellow] (2cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     \\[-2ex]\begin{tikzpicture} 
%       \draw[fill=blue] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=green] (0cm, 0cm) rectangle (0cm, .2cm);
%       \draw[fill=yellow] (0cm, 0cm) rectangle (2cm, .2cm);
%       \draw[fill=black] (2cm, 0cm) rectangle (2cm, .2cm);
%     \end{tikzpicture}
%     }\\ \hline 
%   \end{tabular}
%   \caption{Colors indicate IA from islands A \crule[blue]{.5cm}{.3cm},
%     B \crule[green]{.5cm}{.3cm} and C
%     \crule[yellow]{.5cm}{.3cm}. Here, in the model where only
%     $A\leftrightarrow B$ and $B\leftrightarrow C$ exchange migrants
%     (but not $A\not\leftrightarrow C$), admixture has two sources of
%     errors when estimating IA for a recently admixed individual from
%     $A\times C$. In individual~1, $C$ is overrepresented in the IA-vector
%     $q$. In individual~2, admixture gives the greatest fraction of
%     ancestry to island $B$, since allele frequencies in $B$ are
%     between $A$ and $C$. In both cases, the IA in terms of the
%     combined vector $q^{MP} = (q^M + q^P)/2$ from the recent-admixture
%     model gives more accurate estimates.}
%   \label{fig:sim1}
% \end{figure}

\subsection{Estimation accuracy}
For comparing the accuracy of the admixture and recent-admixture
model, we extended our analysis of the simuated data to the 1000
genomes dataset. We excluded all Admixed Americans (AMRs) since they
are known to have an admixed background \cite{article}{Eduardoff2016,
  Pfaffelhuber2019} and do not form a well-defined own group. As the
true IA, we use the continental origins as described in the dataset,
i.e.\ we have AFR (African), EAS (East-Asia), EUR (European) and SAS
(South Asian) samples. This means e.g.\ that we set
$q^{\text{\sc true}}_{\text{EUR}}=1$ for a European sample.

We ran three kinds of analyses. First, on the non-admixed samples,
i.e.\ the original 1000 genomes data (denoted AFR,...). Second, we
produced in silico recently admixed individuals with parents from the
non-admixed samples (denoted AFR$\times$EAS etc.) and ran the analysis
on these samples. Third, the analysis was performed on
second-generation admixed samples, i.e.\ grand-parents were taken from
the non-admixed samples (denoted AFR/EAS$\times$EUR/SAS etc). In the
first case, Figure~\ref{fig:nonmixed_EUROFORGEN} shows that the
resulting errors for the admixture and recent-admixture model are
almost identical. Overall, recent-admixture has a smaller error in
1364 out of 2157 cases, i.e.\ th hypothesis that the error for
recent-admixture is at least as large as for admixture can be rejected
(binomial test, $p<0.001$. % $p<10^{-34}$).
In the second case, Figure~\ref{fig:mixed_cases_EUROFORGEN}(A) shows
clearly that errors for recent-admixture are smaller for all pairs of
continents. More precisely, in 2279 out of 3000 individuals,
recent-admixture is more accurate ($p<0.001$). % $p<10^{-186}$).
Third, for second-generation admixed individuals,
Figure~\ref{fig:mixed_cases_EUROFORGEN}(B) displays errors in the
cases (A)--(G), and shows that again, recent-admixture is more
accurate. Here, recent-admixture outperforms admixture in 15761 out of
27500 cases (resulting in $p<0.001$) % $ p < 10^{-130}$).
A full list of~55 cases is displayed in
Figure~\ref{fig:mixed_allcases_EUROFORGEN} in the SI. The
corresponding results for the Kidd AIMset are similar and found in the
SI.

\begin{figure*}[htb]
  \hspace{3cm} (A) \hspace{8cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_EUROFORGEN.pdf}\vspace{2cm}}
    \hspace{1cm}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_cases_EUROFORGEN.pdf}}
  \end{center}
  \caption{\label{fig:mixed_cases_EUROFORGEN} For all first generation
    admixed samples (A) and second generation admixed samples (B), we
    computed IA from the admixture and recent-admixture model. The
    distance to the true IA is computed as in \eqref{eq:error}. The
    cases in (B) are as described above. }
\end{figure*}

\subsection{Power of the Likelihood-ratio test for recent admixture}
When fixing the maximal $p$-value for significance of the
likelihood-ration test for recent admixture (as described in MM), we
obtain the power of the test for all cases of recent
admixture. Displaying the false positives (i.e.\ positively tested
non-admixed) against true positives (i.e.\ positively tested admixed)
in cases (B)--(G) for all possible values of $p$, we obtain the
Receiver-Operation-Characteristic (ROC) curve \cite{article}{Fawcett2006}. The
optimal curve nearly hits~0 false positives with~100\% true positives
and has an AUC (Area Under the Curve) of~1. As we see in
Figure~\ref{fig:ROC_EUROFORGEN}, the power of the test differs with
the kind of admixture. For first generation admixed (case (B)), one
non-admixed parent (case (E)) and all grand-parents from different
continents (case (G)), the test is nearly perfect in distinguishing
recent-admixture from admixture. If only half of the genome has two
different ancestries (cases (D) and (F)), the power is reduced. If the
individual is not recently-admixed in first generation, but both
parents are (case (C)), power drops even more. In fact, the latter
case is not recent-admixture as in our definition, since $q^M = q^P$
should technically hold. For the overall performance of the test, we
give some examples in Table~\ref{tab:power}, in particular the power
at $p=0.001$ and AUC in all cases. Results for the EUROFORGEN AIMset
are marginally better than for the Kidd AIMset, which are given in the
SI in Figure~\ref{fig:ROC_Kidd} and Table~\ref{tab:power_Kidd}.

\begin{figure*}[htb]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{roc-curve-EUROFORGENE.pdf}
  \end{center}
  \caption{Using the EUROFORGEN AIMset, we plot false positives (i.e.\
    positive non-admixed individuals, as in case (A) above, againts
    positives for all cases of admixture in second generation.}
  \label{fig:ROC_EUROFORGEN}
\end{figure*}

\begin{table}
  \centering
  \input{power_table_EUROFORGEN.tex}
  \caption{Using the same data as in Figure~\ref{fig:ROC_EUROFORGEN},
    we e.g.\ see that the test for recent admixture turns out to have
    a $p$-value below~1\% in 94\% cases of first generation admixed
    individuals.}
  \label{tab:power}
\end{table}

\subsection{Recent admixture in the 1000 genomes dataset}
From the 1000 genomes dataset, we highlight individuals which give
significant results for the test of recent admixture for both
AIMsets. We exclude AMRs from our analysis, and give in
Table~\ref{fig:1000r-ad} the eight most extreme cases, which show
highly significant results for recent admixture for both
AIMsets. There are six Africans from population ASW (Americans from
Southwest USA), and two from South Asia, one from GIH (Gujarati Indian
from Houston, Texas) and one from BEB (Bengali from Bangladesh). We
note that it is known that the ASW population is admixed
\cite{article}{Eduardoff2016}, but until now, it has not been tested if
admixture is recent.

\begin{itemize}
\item NA20278: Giving the most significant results for both datasets,
  this male has most probably parents of African and European
  ancestry. Note also that $q^{MP}$ and $q$ are very similar for both
  AIMsets.
\item NA20342, NA19625, NA20355: Clearly, one parent has African
  ancestry. The other parent is most likely partly European.
\item NA20274: Our test indicates two parents of different ancestry,
  one mostly African, the other mostly East-Asian.
\item NA20299: Interestingly, the results for both AIMsets differ in
  this example. One parent has most likely African ancestry, the other
  is European according to the EUROFORGEN AIMset and South Asian
  according to the Kidd AIMset.
\item HG03803: Most likely, one parent is of South Asian, the other
  has East-Esian ancestry.
\item NA20864: Most likely, one parent is of South Asian, the other
  has European ancestry.
\end{itemize}

In order to get a clearer picture about {\em true} ancestries in these
individuals, we examined local ancestry in the genome. xxx


\begin{figure}[p]
  \input{outlier.tex}
  \caption{The most extreme individuals in the 1000 genomes dataset in
    terms of a signal for recent admixture. For all individuals, we
    give IA from the admixture model (ad), given by $q$, and the
    recent-admixture model (r-ad), given by
    $q^{MP} = \tfrac 12 (q^M + q^P)$. In this case, we also give PIA,
    given by $q^M, q^P$. We use both, the EUROFORGEN and Kidd
    AIMset. Difference in log-likelihoods for both models is given by
    $\Delta\ell$; see \eqref{eq:DeltaEll}. Colors are AFR
    \crule[red]{.5cm}{.3cm}, EAS \crule[blue]{.5cm}{.3cm}, EUR
    \crule[green]{.5cm}{.3cm}, SAS \crule[yellow]{.5cm}{.3cm}.  }
  \label{fig:1000r-ad}
\end{figure}

\subsection{Runtimes} 
The result is that the analysis is fast. In particular, it scales
linearly with the number of analysed traces. E.g.\ once allele
frequencies for all AIMs from the reference dataset are given,the
$500\cdot 55 = 27500$ individuals which need to be analysed for
Figure~\ref{fig:mixed_cases_EUROFORGEN}, and which are created in
silico, takes about 1.5 seconds per individuals on a standard laptop
computer using the statistical language R.

\section{Discussion}
xxx add references, write some more paragraphs

xxx Mention \cite{article}{Prestes2016}.

We only need allele frequencies. These could e.g. be looked up in
Frog-kb. No full reference dataset is necessary.






We address the inference of recent admixture in estimating individual
admixture (IA), when data from Ancestry Informative Markers is
available. The well-known admixture model assumes that in each
individual for some $q = (q_k)_{k=1,...,K}$, every allele has ancestry
within population $k$ with probability $q_k$. We extend this model by
distinguishing between maternally and paternally inherited alleles
(but still consider only autosomes). The maternally inherited alleles
come with IA $q^M$, and the paternally inherited alleles with $q^P$,
which extends the admixture model if $q^M \neq q^P$. Within this
model, $q^P$ and $q^M$ can be estimated by the same methods as in the
admixture model. Taking the average of $q^M$ and $q^P$, we obtain IA
which is highly similar to $q$ in the admixture model in non-admixed
and admixed individuals. In addition, the IA estimated from the
recent-admixture model in recently admixed individuals, in particular
if the two parents come from different populations, is more accurate
than for the admixture model. Moreover, and in contrast to the
admixture model, we also estimate the form of recent admixture. A
resulting likelihood ratio test for recent admixture has high power to
detect recent admixture in many relevant scenarios, in particular if
the two parents come from different populations.

In many applications, {\sc structure} still seems to be the
gold-standard for estimating the IA of traces. {\sc structure} uses
the same admixture model as we do, it is computationally much more
demanding. The same holds for {\sc admixture} and {\sc frappe}. The
reason is that these programs aim to simultaneously estimate the IA of
the new trace, as well as allele frequencies in ancestral
populations. In other words, in these programs, the new trace changes
allelic frequencies in ancestral populations during the
runtime. However, as in forensics we mostly have a large reference
database and only one (or a few) new traces, the impact on the allelic
frequencies are negligible. Therefore, we take the computationally
easier way and take allele frequencies for ancestral populations as
fixed, and only change the IAs of the new traces during the
runtime. The result is that the analysis is fast. E.g.\ the
$500\cdot 55 = 27500$ individuals which need to be analysed for
Figure~\ref{fig:mixed_cases_EUROFORGEN}, and which are created in
silico, takes about five hours on a standard laptop computer using the
statistical language R.



Not included the GDA from Cheung 2019. In addition, we also only need
frequencies from the reference database (or some external source,
e.g. from Frog-kb), although we use the same model as
structure. Reason is that we do not update frequencies. Updating
frequencies would mean that the trace can alter ancestral allelic
frequencies.


\bibliographystyle{article}{plain}
\bibliography{article}{BGA}{References}
%\bibliographystyle{chicago} \bibliography{BGA}

\newpage
\setcounter{page}{1}
\setcounter{section}{0}
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{equation}{0}
\thispagestyle{empty}

\begin{center}
 \title{\LARGE Supporting Information: Inference
  of recent admixture using genotype data}

~~

\author{\sc Peter Pfaffelhuber, Elisabeth Huss, Franz Baumdicker, \\
  \sc Denise Syndercombe Court, Fabian Staubach}

~

\date{\today}

\maketitle
  
\end{center}

\renewcommand{\theequation}{S\arabic{equation}}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\thesection}{S\arabic{section}}


\section{Theory}
We write down the admixture model and derive a method to estimate
Individual Admixture (IA) in the case when allele frequencies within
populations are not updated. In addition, we give the recent-admixture
model, where an individual is allowed to have parents with different
admixture proportions. We take the following notation for the
reference database:
\begin{align*}
  K: & \text{ number of ancestral populations,}
  \\ M: & \text{ number of markers,}
  \\ p_{mk}: & \text{ frequency of allele 1 at (bi-allelic) marker $m$ in population $k$.}
\end{align*}
In addition, we consider one additional diploid genome (called the
trace) $(G_{m1}, G_{m2})_{m=1,...,M}$, or $(G_m)_{m=1,...,M}$ with
$G_m = G_{m1} + G_{m2}$ if phase is not known. The goal is to estimate
admixture proportions $(q_k)_{k=1,...,K}$ (or
$(q^M_{k})_{k=1,...,K}, (q^P_{k})_{k=1,...,K}$) of this additional
genome.  Recall that we take the approach from
\cite{SI}{Chakraborty1986} and \cite{SI}{Hanis1986} and do
not update $p_{mk}$'s during the analysis.

\subsection{The admixture model}
In \cite{SI}{Alexander2009}, \cite{SI}{Pritchard2000},
\cite{SI}{Tang2005} and elsewhere, the main goal is to maximize the
log-likelihood (see also (1) and (2) of \cite{SI}{Tang2005})
\begin{align*}
  \ell(q|G) = \sum_{m=1}^M \sum_{a=1,2} \log\Big(\sum_{k=1}^K \alpha_{mak}q_k\Big),  
\end{align*}
where
$$\alpha_{mak} := \begin{cases} p_{mk}, & \text{ if }G_{ma}=1,\\ 1-p_{mk}, & \text{ if }G_{ma}=0.\end{cases}$$
is the frequency of the observed allele in copy $a$ of marker $m$ in
population $k$. (Note that $e^{\ell(q|G)}$ is the probabiltiy of
observing $(G_{ma})_{m=1,...,M; a=1,2}$, if every allele is picked
independently from population $k$with probability $q_k$.) Assuming
that phase is not known, and with
\begin{align}
  \label{eq:amkl}
  \alpha_{mkl}:= \alpha_{m1k}\alpha_{m2l} = \begin{cases}
    p_{mk}p_{ml}, & \text{ if } G_{m1} + G_{m2} = 2, \\
    p_{mk}(1-p_{ml}) + (1-p_{mk})p_{ml}, & \text{ if } G_{m1} + G_{m2} = 1, \\
    (1-p_{mk})(1-p_{ml}), & \text{ if } G_{m1} + G_{m2} =
    0, \end{cases}
\end{align}
note that the log-likelihood can as well be written as
\begin{align*}
  \ell(q|G) = \sum_{m=1}^M \log\Big(\sum_{k,l=1}^K \alpha_{mkl}q_kq_l\Big).
\end{align*}
We set $\beta_m(q) := \sum_{k=1}^K p_{mk}q_k$ and analyse the last sum
by distinguishing the case $G_m=2$, where
\begin{align*}
  \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = \Big(\sum_{k=1}^kp_{mk}q_k\Big)^2 = \beta_m(q)^2
    \intertext{while for $G_m=1$ we find}
    \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = 2\Big(\sum_{k=1}^kp_{mk}q_k\Big)\Big(1-\sum_{k=1}^kp_{mk}q_k\Big) = 2\beta_m(q)(1-\beta_m(q))
    \intertext{and for $G_m=0$}
    \sum_{k,l=1}^K \alpha_{mkl}q_kq_l
  & = \Big(1-\sum_{k=1}^kp_{mk}q_k\Big)^2 = (1-\beta_m(q))^2,
\end{align*}
such that
\begin{align}  \label{Seq:lik}
  \ell(q|G) = \sum_{m=1}^M \log\Big(\binom{2}{G_m} \beta_m(q)^{G_m}(1-\beta_m(q))^{2-G_m}\Big)
\end{align}

\begin{lemma}\label{l1}
  The maximum of $q\mapsto \ell(q|G)$ under the constraint
  $\sum_{k=1}^K q_k = 1$ solves
  \begin{align}\label{eq:0}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K
    \alpha_{mk'l'}q_{k'}q_{l'}} = 1, \qquad k=1,...,K.
  \end{align}
\end{lemma}

\begin{remark}
  \begin{enumerate}
  \item Assume we add a set of {\em Ancestry Uninformative Markers},
    i.e.\ a set of markers $\tilde m = 1,...,\tilde M$, with
    frequencies not depending on the population, i.e.\
    $\alpha_{\tilde m k l} = \alpha_{\tilde m k'l'}$ for
    $k,l,k,l''=1,...,K$. For these markers,
    $$\sum_{\tilde m = 1}^{\tilde M} \sum_{l=1}^K \frac{\alpha_{\tilde
        mkl}q_l}{\sum_{k',l'=1}^K \alpha_{\tilde mk'l'}q_{k'}q_{l'}} =
    \tilde M.$$ This implies that $q$ is a solution of \eqref{eq:0}
    without these markers iff $q$ is a solution of \eqref{eq:0} if
    these markers are included. This might be reassuring.
  \item Let us have a closer look at the left hand side of
    \eqref{eq:0}. We set $\beta_m := \sum_{k=1}^K p_{mk} q_k$. For
    $G_m=2$,
    \begin{align*}
      \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & = 
        \sum_{l=1}^K \frac{p_{mk}p_{ml}q_l}{\sum_{k',l'}p_{mk'}p_{ml'}q_{k'}q_{l'}} = \frac{p_{mk}}{\beta_m}.
        \intertext{For $G_m=1$, we have}
        \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & =
        \sum_{l=1}^K \frac{(p_{mk}(1-p_{ml}) + (1-p_{mk})p_{ml})q_l}{\sum_{k',l'}(p_{mk'}(1-p_{ml'})
        + (1-p_{mk'})p_{ml'})q_{k'}q_{l'}}
      \\ & = \frac{p_{mk}(1-\beta_m) + (1-p_{mk})\beta_m}{2\beta_m(1-\beta_m)}
           = \frac 12\Big(\frac{p_{mk}}{\beta_m} + \frac{1-p_{mk}}{1-\beta_m}\Big)
           \intertext{and for $G_m=0$}
           \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'}\alpha_{mk'l'}q_{k'}q_{l'}}
      & = 
        \sum_{l=1}^K \frac{(1-p_{mk})(1-p_{ml})q_l}{\sum_{k',l'}(1-p_{mk'})(1-p_{ml'})q_{k'}q_{l'}} = \frac{1-p_{mk}}{1-\beta_m}.
    \end{align*}
    In total, this gives that $q$ needs to solve
    \begin{align*}
      \frac{1}{2M} \sum_{m=1}^M \Big(G_m \frac{p_{mk}}{\beta_m} + (2-G_m)\frac{1-p_{mk}}{1-\beta_m}\Big) = 1
    \end{align*}
    In order to find a solution, we reformulate as the fixed point
    equation
    \begin{align}\label{eqSI:fixed}
      q_k = f_k(q) \text{ for }f_k(q) =
      \frac{1}{2M} \sum_{m=1}^M \Big(G_m \frac{p_{mk}}{\beta_m} + (2-G_m)\frac{1-p_{mk}}{1-\beta_m}\Big)q_k,
      \qquad k =1,...,K.
    \end{align}
    A solution can be computed by iteratively computing
    $q' = (f_k(q))_{k=1,...,K}$.
  \end{enumerate}
\end{remark}

\begin{proof}[Proof of Lemma~\ref{l1}]
  We use the theory of Lagrange multipliers, since we need to maximize
  $\ell$ over $q$ under the constraint $\sum_k q_k=1$.  Since
  \begin{align*}
    \frac{\partial \ell(q|G)}{\partial q_k} & = \sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K
                                              \alpha_{mk'l'}q_{k'}q_{l'}},
  \end{align*}
  we have to solve the system of equations
  \begin{align*}
    \lambda & = \sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}}, \qquad k=1,...,K,
    \\
    1 & = \sum_{k=1}^K q_k.
  \end{align*}
  It is easy to eliminate $\lambda$, since the last equation gives
  \begin{align*}
    \lambda & = \lambda \sum_{k=1}^K q_k = \sum_{m=1}^M \sum_{k,l=1}^K
              \frac{\alpha_{mkl}q_kq_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}} = M.
  \end{align*}
  So, we are left with finding $q = (q_k)_{k=1,...,K}$ such that
  \begin{align*}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K \frac{\alpha_{mkl}q_l}{\sum_{k',l'=1}^K \alpha_{mk'l'}q_{k'}q_{l'}} = 1, \qquad k=1,...,K.
  \end{align*}
\end{proof}


\subsection{The recent-admixture model}
\noindent
For the recent-admixture-version, we want to estimate
$q^M=(q^M_{k})_{k=1,...,K}, q^P=(q^P_{k})_{k=1,...,K}$, where
$q^M_{k}$ and $q^P_k$ are the fractions of the genomes of the mother
and father, respectively, which come from population $k$.  This
assumption implies that the log-likelihood is
$$ \ell(q^M, q^P|G) = \sum_{m=1}^M  \log\Big(\sum_{k,l=1}^K \alpha_{mkl}q_{k}^M q_l^P\Big),$$
where $\alpha_{mkl}$ is given as in \eqref{eq:amkl}.  As is apparent
from the log-Likelihood function, the recent-admixture model
generalizes the admixture model. Put differently, choosing
$q^M = q^P = q$ in the recent-admixture model gives the admixture
model. We note that again, the log-likelihood can be written
differently,
\begin{equation}
  \begin{aligned}\label{eqSI:logLrecentadmixture}
    \ell(q^M, q^P|G) & = \sum_{m=1}^M  \log\Big( 1_{G_m=2} \beta_m(q^M) \beta_m(q^P)
    \\ & \qquad \qquad \qquad + 1_{G_m=1}(\beta_m(q^M) (1-\beta_m(q^P))
    + (1-\beta_m(q^M)) \beta_m(q^P))
    \\ & \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad + 1_{G_m=0}(1-\beta_m(q^M))(1-\beta_m(q^P))\Big).    
  \end{aligned}
\end{equation}

\begin{lemma}\label{l2}
  The maximum of $(q^M, q^P)\mapsto \ell(q^M, q^P,G)$ under the
  constraint $\sum_{k=1}^K q_k^M = \sum_{k=1}^K q_k^P = 1$ solves
  \begin{align}\label{eq:0recent}
    \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
    = \frac{1}{M}\sum_{m=1}^M \sum_{l=1}^K\frac{\alpha_{mkl}q_l^P}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^P_{k'}q^M_{l'}} = 1, \qquad k=1,...,K.
  \end{align}
\end{lemma}

% \begin{align*}
%   & p_{mk}\Big(\frac{1-\beta_m(q^M)}{\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P)}
%     - \frac{1}{\beta_m(q^P)}\Big)
%   \\ & \qquad + (1-p_{mk})
%        \Big(\frac{\beta_m(q^M)}{\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P)} - \frac{1}{1-\beta_m(q^P)}\Big)
%   \\ & = - p_{mk}\frac{\beta_m(q^M)(1-\beta_m(q^P))}{\beta_m(q^P)(\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P))}
%   \\ & \qquad - (1-p_{mk})\frac{(1-\beta_m(q^M))\beta_m(q^P)}{(1-\beta_m(q^P))(\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P))}
% \end{align*}


\begin{remark}
  \begin{enumerate}
  \item Note that \eqref{eq:0recent} is symmetric in $q^M$ and $q^P$,
    i.e.\ if $(q^M, q^P)$ solve \eqref{eq:0recent}, another solution
    is given by $(q^P, q^M)$.
  \item Again, we can turn \eqref{eq:0recent} into fixed point
    equations. To derive it, we again have a closer look at the left
    hand side of \eqref{eq:0recent}. For
    $\beta_m(q) := \sum_k p_{mk} q_k$, we have for $G_m=2$
    \begin{align*}
      \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{p_{mk} \beta_m(q^M)}{\beta_m(q^M)\beta_m(q^P)} = \frac{p_{mk}}{\beta_m(q^P)},
        \intertext{for $G_m=1$}
        \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{p_{mk} (1-\beta_m(q^M)) + (1-p_{mk})\beta_m(q^M)}{\beta_m(q^M)(1-\beta_m(q^P)) + (1-\beta_m(q^M))\beta_m(q^P)}
        \intertext{and for $G_m=0$}
        \sum_{l=1}^K  \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'=1}^K \alpha_{mk'l'}q^M_{k'}q^P_{l'}}
      &
        = \frac{(1-p_{mk})(1- \beta_m(q^M))}{(1-\beta_m(q^M))(1-\beta_m(q^P))} = \frac{1-p_{mk}}{1-\beta_m(q^P)}.
    \end{align*}
    So, we suggest to iteratively compute
    \begin{align}\label{eqSI:fixedRecent}
      \tilde q^P & = f(q^M, q^P) \text{ and } \tilde q^M = f(\tilde q^P, q^M)
                   \intertext{for $f(q, q') = (f_k(q, q'))_{k=1,...,K}$ with} \notag
                   f_k(q, q') & := \frac{1}{M}\sum_{m=1}^M \Big(1_{G_m=2} \frac{p_{mk}}{\beta_m(q')}
      \\ \notag & \qquad \qquad \qquad + 1_{G_m=1} \frac{(p_{mk} (1-\beta_m(q))
                  + (1-p_{mk})\beta_m(q))}{\beta_m(q)(1-\beta_m(q'))
                  + (1-\beta_m(q))\beta_m(q')}
      \\ \notag & \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad \qquad
                  + 1_{G_m=0} \frac{(1-p_{mk})}{1-\beta_m(q')}\Big)q_k'.
    \end{align}
  \end{enumerate}
\end{remark}

\begin{proof}[Proof of Lemma~\ref{l2}]
  Again, we use Lagrange multipliers. Since
  \begin{align*}
    \frac{\partial \ell(q^M, q^P|G)}{\partial q^P_k}
    & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}},
    \\
    \frac{\partial \ell(q^M, q^P|G)}{\partial q^M_k}
    & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^P}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}},
  \end{align*}
  we have to solve the system of equations
  \begin{align*}
    \lambda & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}}, \qquad k=1,...,K,\\
    \rho & = \sum_{m=1}^M \sum_l \frac{\alpha_{mak}q_l^P}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q^M_{k'}}, \qquad k=1,...,K,
    \\
    1 & = \sum_{k=1}^K q^P_k = \sum_{k=1}^K q^M_k.
  \end{align*}
  It is easy to eliminate $\lambda$ and $\rho$, since
  \begin{align*}
    \lambda & = \lambda \sum_{k=1}^K q^P_k = \sum_{m=1}^M \sum_{k,l}
              \frac{\alpha_{mkl}q^P_kq_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q_{l'}^M} = M,
    \\
    \rho & = \rho \sum_{k=1}^K q_k^M = \sum_{m=1}^M \sum_{k,l}
           \frac{\alpha_{mkl}q^P_kq_l^M}{\sum_{k',l'} \alpha_{mk'l'}q^P_{k'}q_{l'}^M} = M.    
  \end{align*}
  So, we are left with finding $q^P$ and $q^M$ such that
  \begin{align*}
    \frac{1}{M}\sum_{m=1}^M \frac{\alpha_{mkl}q_l^P}{\sum_{k',l'}^K \alpha_{mk'l'}q^P_{k'} q^M_{l'}} & = 1, \qquad k=1,...,K,
    \\ \frac{1}{M}\sum_{m=1}^M \frac{\alpha_{mkl}q_l^M}{\sum_{k',l'}^K \alpha_{mk'l'}q^P_{k'} q^M_{l'}} & = 1, \qquad k=1,...,K.
  \end{align*}
\end{proof}

\newpage

\section{Additional results}

\subsection{Some showcases from simulations}
\enlargethispage{0cm}
\begin{figure}[H]
  \begin{center}
    \parbox[b]{0.9\textwidth}{\includegraphics[width=0.95\textwidth]{abc_StructurePlot.pdf}\vspace{-2cm}}
  \end{center}
  \caption{\label{Sfig:sim2} Structure barplot for all 400 recently
    $A\times C$-admixed individuals. Arrows indicate the two
    individuals from Figure~\ref{fig:sim1}.}
\end{figure}

\begin{figure}[H]
  \begin{center}
    \parbox[b]{0.5\textwidth}{\includegraphics[width=0.65\textwidth]{deviations_abc.pdf}\vspace{0cm}}
  \end{center}
  \caption{\label{Sfig:sim3} For all 3$\times$400 non-admixed samples,
    and 3$\times$400 admixed samples, we compute the errors by the
    distance to the true IA as given in \eqref{eq:error}.}
\end{figure}

\newpage

\subsection{Estimation accuracy}
For first generation admixed individuals, all cases (non-admixed and
parents come from different continents) are described in the main
text. For second generation admixed individuals, we have several
cases, depending on the origin of the grand-parents.  When data from
four populations (AFR, EAS, EUR, SAS; see below) is available, we have
the following cases:
\begin{itemize}
\item[(A)] 4 non-admixed cases with IA 100:0: \\
  AFR/AFR$\times$AFR/AFR, EAS/EAS$\times$EAS/EAS,
  EUR/EUR$\times$EUR/EUR, SAS/SAS$\times$SAS/SAS;
\item[(B)] 6 admixed cases with admixture ratio 50:50, where both
  parents are non-admixed:\\
  AFR/AFR$\times$EAS/EAS, AFR/AFR$\times$EUR/EUR,
  AFR/AFR$\times$SAS/SAS, EAS/EAS$\times$EUR/EUR,
  EAS/EAS$\times$SAS/SAS, EUR/EUR$\times$SAS/SAS;
\item[(C)] 6 admixed cases with admixture ratio 50:50, where both
  parents are admixed:\\
  AFR/EAS$\times$AFR/EAS, AFR/EUR$\times$AFR/EUR,
  AFR/SAS$\times$AFR/SAS, EAS/EUR$\times$EAS/EUR,
  EAS/SAS$\times$EAS/SAS, EUR/SAS$\times$EUR/SAS;
\item[(D)] 12 admixed cases with admixture ratio 75:25:\\
  AFR/AFR$\!\times$AFR/EAS, AFR/AFR$\times$AFR/EUR,
  AFR/AFR$\!\times$AFR/SAS, EAS/EAS$\times$EAS/AFR,
  EAS/EAS$\!\times$EAS/EUR, EAS/EAS$\times$EAS/SAS,
  EUR/EUR$\!\times$EUR/AFR, EUR/EUR$\times$EUR/EAS,
  EUR/EUR$\!\times$EUR/SAS, SAS/SAS$\times$SAS/AFR,
  SAS/SAS$\!\times$SAS/EAS, SAS/SAS$\times$SAS/EUR;
\item[(E)] 12 second generation admixed with admixture ratio 50:25:25,
  where one parent is non-admixed:\\
  AFR/AFR$\times$EAS/EUR, AFR/AFR$\times$EAS/SAS,
  AFR/AFR$\times$EUR/SAS, EAS/EAS$\times$AFR/EUR,
  EAS/EAS$\times$AFR/SAS, EAS/EAS$\times$EUR/SAS,
  EUR/EUR$\times$AFR/EAS, EUR/EUR$\times$AFR/SAS,
  EUR/EUR$\times$EAS/SAS, SAS/SAS$\times$AFR/EAS,
  SAS/SAS$\times$AFR/EUR, SAS/SAS$\times$EAS/EUR;
\item[(F)] 12 second generation admixed with admixture ratio 50:25:25,
  where both parents are admixed:\\
  AFR/EAS$\times$AFR/EUR, AFR/EAS$\times$AFR/SAS,
  AFR/EUR$\times$AFR/SAS, EAS/AFR$\times$EAS/EUR,
  EAS/AFR$\times$EAS/SAS, EAS/EUR$\times$EAS/SAS,
  EUR/AFR$\times$EUR/EAS, EUR/AFR$\times$EUR/SAS,
  EUR/EAS$\times$EUR/SAS, SAS/AFR$\times$SAS/EAS,
  SAS/AFR$\times$SAS/EUR, SAS/EAS$\times$SAS/EUR;
\item[(G)] 3 second generation admixed with admixture ratio
  25:25:25:25:\\
  AFR/EAS$\times$EUR/SAS, AFR/EUR$\times$EAS/SAS,
  AFR/SAS$\times$EAS/EUR;
\end{itemize}
As can be seen in Figure~\ref{fig:mixed_allcases_EUROFORGEN}, the
recent-admixture model never gives worse estimates than the admixture
model, and outperforms the admixture-model in several cases; see also
Figure~\ref{fig:mixed_cases_EUROFORGEN} in the main text.  For the
Kidd AIMset, see Figures~\ref{fig:mixed_cases_Kidd}
and~\ref{fig:mixed_allcases_Kidd}.

\subsubsection*{EUROFORGEN AIMset}

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{deviations_nonmixed_EUROFORGEN.pdf}
  \end{center}
  \caption{\label{fig:nonmixed_EUROFORGEN} For all non-admixed samples
    from the 1000 genomes dataset, we computed IA from the admixture
    and recent-admixture model. The distance to the true IA is
    computed as in \eqref{eq:error}.}
\end{figure}

\begin{center}
  \begin{turn}{270}%[htbp]
    \begin{minipage}{\textheight}
      \includegraphics[width=\textwidth]{deviations_mixed_allcases_EUROFORGEN.pdf}
      \captionof{figure}{For all cases of second generation admixed individuals, we
        compare estimation accuracy of IA using the EUROFORGEN AIMset.}
      \label{fig:mixed_allcases_EUROFORGEN}
    \end{minipage}
  \end{turn}  
\end{center}
       
\subsubsection*{Kidd AIMset}
For the Kidd AIMset, recent-admixture produces an error in the
non-admixed samples (see also Figure~\ref{fig:nonmixed_Kidd}), which
is smaller than the error for the admixture-model in 1394 out of 2157
cases, so the hypothesis that the error is worse for recent-admixture
can be rejected ($p<0.001$). % $ p < 10^{-34}$).
For first order admixed samples, 2381 out of 3000 cases have a smaller
error under recent-admixture (which gives $p<0.001$; %$p<10^{-241}$);
see also Figure~\ref{fig:mixed_cases_Kidd}.A. Last, for
second-generation admixed individuals, recent-admixture is more
accurate in 16116 out of 27500 cases ($p<0.001$); % $p<10^{-180}$);
see also Figures~\ref{fig:mixed_cases_Kidd}.B and
\ref{fig:mixed_allcases_Kidd}.

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{deviations_nonmixed_Kidd.pdf}
  \end{center}
  \caption{\label{fig:nonmixed_Kidd} Same as in
    Figure~\ref{fig:nonmixed_EUROFORGEN}, but for the Kidd AIMset.}
\end{figure}



\begin{figure}[H]
  \hspace{3cm} (A) \hspace{8cm} (B)
  \begin{center}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_Kidd.pdf}\vspace{2cm}}
    \hspace{1cm}
    \parbox[b]{0.45\textwidth}{\includegraphics[width=0.45\textwidth]{deviations_mixed_cases_Kidd.pdf}}
  \end{center}
  \caption{\label{fig:mixed_cases_Kidd} Same as in
    Figure~\ref{fig:mixed_cases_EUROFORGEN}, but using the Kidd
    AIMset.}
\end{figure}

\begin{table}[H]
  \centering
  \input{power_table_Kidd.tex}
  \caption{Same as Table~\ref{tab:power}, but using the Kidd AIMset.}
  \label{tab:power_Kidd}
\end{table}

\begin{center}
  \begin{turn}{270}%[htbp]
    \begin{minipage}{\textheight}
      \includegraphics[width=\textwidth]{deviations_mixed_allcases_Kidd.pdf}
      \captionof{figure}{Same as in
        Figure~\ref{fig:mixed_allcases_EUROFORGEN}, but using the Kidd
        AIMset.}
      \label{fig:mixed_allcases_Kidd}
    \end{minipage}
  \end{turn}
\end{center}

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=0.45\textwidth]{roc-curve-Kidd.pdf}
  \end{center}
  \caption{Same as in Figure~\ref{fig:ROC_EUROFORGEN}, but using the
    Kidd AIMset.}
  \label{fig:ROC_Kidd} 
\end{figure}

\bibliographystyle{SI}{plain}
\bibliography{SI}{BGA}{References}

\end{document}



\documentclass{article}
\usepackage{tikz}
\usetikzlibrary{shapes,arrows,positioning}
\tikzstyle{line}=[draw]

\begin{document}

\begin{center}
\begin{tikzpicture}
%U = unaffected 
%A = affected 
%C = carrier 
%M = male 
%F = female 
[scale=1,every node/.style={transform shape},
  UM/.style={rectangle, draw=black,very thick, minimum size=5mm,minimum width = 5mm,minimum height = 5mm},
  UF/.style={circle,draw = black, very thick, minimum size=5mm,minimum width = 5mm,minimum height = 5mm},
  AM/.style={rectangle, fill=black,very thick, minimum size=5mm,minimum width = 5mm},
  AF/.style={circle,fill=black,very thick, minimum size=5mm,minimum width = 5mm},
  CM/.style={rectangle, left color = white, right color = black, draw=black,very thick, minimum size=5mm,minimum width = 5mm,minimum height = 5mm},
  CF/.style={circle, left color = white, right color = black,draw = black, very thick, minimum size=5mm,minimum width = 5mm,minimum height = 5mm}
]

%parents
\node[UM](dad){};
\node[CF,xshift = 30pt](mom) at (dad.east){};
\draw[line] (dad) -- (mom);

%kids
\node[AM,yshift=-30pt](kid1) at (dad.south) {};
\node[UF,yshift=-30pt](kid2) at (mom.south) {};
\node[UM,xshift=30pt](kid3) at (kid2.east){};
\node[UF,xshift=-30pt](kid4) at (kid1.west){};
\draw[line] (kid1) --(kid2);
\draw[line] (kid2) --(kid3);
\draw[line] (kid1) --(kid4);


%connect parents to kids
\path[draw = black] (0.75,0) -- (0.75,-1.35);

\end{tikzpicture}
\end{center}

\end{document}